<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.340">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>ENaC isn’t ASIC - Appendix A — Materials and Methods</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/appendices/processing-flow.html" rel="next">
<link href="../../parts/references.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../figures/other/domain-cartoon_ECD.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">ENaC isn’t ASIC</span>
    </a>
  </div>
        <div class="quarto-navbar-tools ms-auto">
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.pdf">
              <i class="bi bi-bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.docx">
              <i class="bi bi-bi-file-word pe-1"></i>
            Download Docx
            </a>
          </li>
      </ul>
    </div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/appendices/mat-meth.html">Appendices</a></li><li class="breadcrumb-item"><a href="../../parts/appendices/mat-meth.html"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Materials and Methods</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Frontmatter</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_enac-in-the-body.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">ENaC At The Organism Scale</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Results and Discussion</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_activating-conditions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_transmembrane-domain.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">The Transmembrane Domain Remains Elusive</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/mat-meth.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Materials and Methods</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/processing-flow.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Image Processing</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/validation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Map Validation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/code.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Code and Computation</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#enac-expression" id="toc-enac-expression" class="nav-link active" data-scroll-target="#enac-expression"><span class="header-section-number">A.1</span> ENaC Expression</a>
  <ul class="collapse">
  <li><a href="#human-enac" id="toc-human-enac" class="nav-link" data-scroll-target="#human-enac"><span class="header-section-number">A.1.1</span> Human ENaC</a></li>
  <li><a href="#mouse-enac" id="toc-mouse-enac" class="nav-link" data-scroll-target="#mouse-enac"><span class="header-section-number">A.1.2</span> Mouse ENaC</a></li>
  </ul></li>
  <li><a href="#enac-purification" id="toc-enac-purification" class="nav-link" data-scroll-target="#enac-purification"><span class="header-section-number">A.2</span> ENaC Purification</a>
  <ul class="collapse">
  <li><a href="#human-enac-1" id="toc-human-enac-1" class="nav-link" data-scroll-target="#human-enac-1"><span class="header-section-number">A.2.1</span> Human ENaC</a></li>
  <li><a href="#mouse-enac-1" id="toc-mouse-enac-1" class="nav-link" data-scroll-target="#mouse-enac-1"><span class="header-section-number">A.2.2</span> Mouse ENaC</a></li>
  </ul></li>
  <li><a href="#enac-nanodisc-preparation" id="toc-enac-nanodisc-preparation" class="nav-link" data-scroll-target="#enac-nanodisc-preparation"><span class="header-section-number">A.3</span> ENaC nanodisc preparation</a></li>
  <li><a href="#western-blots" id="toc-western-blots" class="nav-link" data-scroll-target="#western-blots"><span class="header-section-number">A.4</span> Western Blots</a></li>
  <li><a href="#cryoem-grid-preparation" id="toc-cryoem-grid-preparation" class="nav-link" data-scroll-target="#cryoem-grid-preparation"><span class="header-section-number">A.5</span> cryoEM Grid Preparation</a></li>
  <li><a href="#cryoem-data-collection" id="toc-cryoem-data-collection" class="nav-link" data-scroll-target="#cryoem-data-collection"><span class="header-section-number">A.6</span> cryoEM Data Collection</a></li>
  <li><a href="#cryoem-image-processing" id="toc-cryoem-image-processing" class="nav-link" data-scroll-target="#cryoem-image-processing"><span class="header-section-number">A.7</span> cryoEM Image Processing</a></li>
  <li><a href="#model-building" id="toc-model-building" class="nav-link" data-scroll-target="#model-building"><span class="header-section-number">A.8</span> Model Building</a></li>
  <li><a href="#constructs" id="toc-constructs" class="nav-link" data-scroll-target="#constructs"><span class="header-section-number">A.9</span> Constructs</a></li>
  <li><a href="#whole-cell-patch-clamp" id="toc-whole-cell-patch-clamp" class="nav-link" data-scroll-target="#whole-cell-patch-clamp"><span class="header-section-number">A.10</span> Whole cell patch clamp</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Appendix A — Materials and Methods</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="enac-expression" class="level2" data-number="A.1">
<h2 data-number="A.1" class="anchored" data-anchor-id="enac-expression"><span class="header-section-number">A.1</span> ENaC Expression</h2>
<section id="human-enac" class="level3" data-number="A.1.1">
<h3 data-number="A.1.1" class="anchored" data-anchor-id="human-enac"><span class="header-section-number">A.1.1</span> Human ENaC</h3>
<p>Human embryonic kidney cells (HEK293T/17) were grown in suspension in Freestyle medium with 2% FBS at a density of 2 – 4<span class="math inline">\(\times 10^6\)</span> cells/mL. BacMam virus carrying the genes for the appropriate ENaC subunits were added to the flasks at an MOI of 1 and incubated at 37&nbsp;°C for 8 hours, after which 1 μM amiloride was added and the flasks were moved to 30&nbsp;°C. After a total incubation time of 72 hours, cultures were centrifuged at 4,790 xg for 20 minutes to collect pellets. The pellets were washed with <strong>TBS</strong> (20&nbsp;mM Tris pH&nbsp;8.0, 200&nbsp;mM NaCl) and centrifuged again before snap-freezing in liquid nitrogen. Cell pellets were stored at -80&nbsp;°C until use.</p>
</section>
<section id="mouse-enac" class="level3" data-number="A.1.2">
<h3 data-number="A.1.2" class="anchored" data-anchor-id="mouse-enac"><span class="header-section-number">A.1.2</span> Mouse ENaC</h3>
<p>Sf9 cells were grown in suspension in Sf900-II serum-free media at 27&nbsp;°C. Cells were infected at log phase with virus encoding full-length mouse ENaC<sub>α</sub>, ENaC<sub>β</sub>, and ENaC<sub>γ</sub>. After 5–6 hours of growth, 1&nbsp;μM amiloride was added to the culture. Cells were harvested 24&nbsp;hours later, washed with TBS, flash frozen in liquid nitrogen, and stored at -80&nbsp;°C until used.</p>
</section>
</section>
<section id="enac-purification" class="level2" data-number="A.2">
<h2 data-number="A.2" class="anchored" data-anchor-id="enac-purification"><span class="header-section-number">A.2</span> ENaC Purification</h2>
<section id="human-enac-1" class="level3" data-number="A.2.1">
<h3 data-number="A.2.1" class="anchored" data-anchor-id="human-enac-1"><span class="header-section-number">A.2.1</span> Human ENaC</h3>
<p>Throughout this protocol, quantities of resin and buffer are given “per flask”. I grow cell culture in quantities of 800&nbsp;mL culture per flask, meaning that, e.g., pelleted cells from 1.6 L of cells require twice the quoted quantities of buffer and resin. In datasets with NaCl listed as the salt, solubilization buffer, wash buffers, elution buffer, and SEC buffer used NaCl in place of KCl.</p>
<p>To purify ENaC, frozen whole-cell pellets were removed from the -80&nbsp;°C freezer and thawed in a 37&nbsp;°C water bath. While the pellets were thawing, 50&nbsp;mL <strong>Solubilization Buffer</strong> was prepared per flask:</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;7.5, 200&nbsp;mM KCl</li>
<li>1 tablet per 100&nbsp;mL Pierce EDTA-free protease inhibitor tablets</li>
<li>25 U/mL ThermoFisher universal nuclease</li>
<li>2.5&nbsp;mM ATP, 5&nbsp;mM MgCl<sub>2</sub></li>
<li>10&nbsp;mg/mL digitonin</li>
<li>10 μM amiloride</li>
</ul>
<p>The solubilization buffer was prepared fresh at 2x concentration (i.e., half the final volume). Once pellets were fully thawed, they were added to the solubilization buffer and the mixture was brought to the final volume with DI water and poured into a screw-cap bottle. The solubilization mixture was left at 8&nbsp;°C for 1.5 hours.</p>
<p>After solubilization was complete, the crude lysate was decanted into ultracentrifuge tubes and centrifuged at 100,000 xg for 45 minutes at 4&nbsp;°C. <!-- TODO: where did I get the GNB gene? --> Meanwhile, in-house-made GFP-nanobody CNBr resin (0.8&nbsp;mL per flask) was loaded into an XK 16 column and washed with 2&nbsp;CV of TBK</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;7.5</li>
<li>200&nbsp;mM KCl</li>
</ul>
<p>and the following buffers were prepared in the listed quantities:</p>
<p><strong>wash A</strong> (10 column volumes [CV]):</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;7.5, 200&nbsp;mM KCl</li>
<li>0.7&nbsp;mg/mL digitonin</li>
<li>2&nbsp;mM MgCl<sub>2</sub></li>
<li>2&nbsp;mM ATP</li>
<li>2&nbsp;nM phenamil</li>
</ul>
<p><strong>wash B</strong> (10&nbsp;CV):</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;7.5</li>
<li>200&nbsp;mM KCl</li>
<li>0.7&nbsp;mg/mL digitonin</li>
<li>5&nbsp;mM CaCl<sub>2</sub></li>
<li>2&nbsp;nM phenamil;</li>
</ul>
<p><strong>elution buffer</strong> (5&nbsp;CV): wash B with 33 μg/mL thrombin</p>
<p>The column was washed with 2&nbsp;CV of wash A, then clarified lysate was bound to the column under 4&nbsp;mL/min flow. A small aliquot of pre-column and post-column clarified lysate was reserved to assess binding efficiency by GFP FSEC<span class="citation" data-cites="kawate06"><sup><a href="../references.html#ref-kawate06" role="doc-biblioref">1</a></sup></span>. If binding efficiency fell below 90% of the initial efficiency the resin was discarded. Otherwise, it was regenerated with pulsed pH&nbsp;cleaning. The GFP-nanobody column was washed with 5&nbsp;CV wash A and B.</p>
<p>To elute ENaC from the GFP nanobody resin, the column was washed 3 times with 1.5&nbsp;CV elution buffer and once with TBK with 0.7&nbsp;mg/mL digitonin. As each wash was flowed on, the eluant was collected in 1&nbsp;mL or 0.25 CV fractions, whichever was smaller. This results in four sets of fractions: the first comprises largely wash B with little protein (from the initial flow-on of elution buffer), and the remaining three elution buffer. All fractions were run on an SDS-PAGE gel, and fractions with bands at the expected molecular weight for ENaC were pooled and concentrated to a final volume of 0.5&nbsp;mL. If the preparation was acid-shocked, the protein was desalted into <strong>acid shock</strong> buffer at this stage:</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;6</li>
<li>200&nbsp;mM KCl</li>
<li>0.7&nbsp;mg/mL digitonin</li>
<li>2&nbsp;nM phenamil</li>
</ul>
<p>If the preparation required Fabs, they were added in two-fold excess of protein concentration (as determined by nanodrop A<sub>280</sub>) at this stage. A final concentration of 1&nbsp;mM C8 PIP<sub>2</sub> was added to the CKO/DEG monofab samples just before grid preparation.</p>
<p>The concentrated eluant was loaded onto a Superose 6 Increase 10/300 column (Cytiva) equilibrated in <strong>SEC buffer</strong> for further purification via SEC.</p>
<ul>
<li>20&nbsp;mM Tris pH&nbsp;7.5</li>
<li>200&nbsp;mM KCl</li>
<li>0.7&nbsp;mg/mL digitonin</li>
<li>2&nbsp;nM phenamil</li>
</ul>
<p>SEC was typically run overnight, meaning fractions remained at 4&nbsp;°C until the next morning. Fractions from the SEC peak were collected and assayed for purity and monodispersity by SDS-PAGE and tryptophan FSEC. Monodisperse peaks were pooled and used the same day. Thus, total time between cell pellet thaw and protein use was typically less than 36 hours.</p>
</section>
<section id="mouse-enac-1" class="level3" data-number="A.2.2">
<h3 data-number="A.2.2" class="anchored" data-anchor-id="mouse-enac-1"><span class="header-section-number">A.2.2</span> Mouse ENaC</h3>
<p>Frozen cell pellet was thawed on ice. Membranes were prepared by sonication and centrifugation, then re-frozen until purification. For each gram of membranes, 15&nbsp;mL of <strong>solubilization buffer</strong> was added</p>
<ul>
<li>4&nbsp;mM GDN</li>
<li>20&nbsp;mM HEPES pH&nbsp;7.6</li>
<li>150&nbsp;mM NaCl</li>
<li>2&nbsp;mM ATP</li>
<li>2&nbsp;mM MgSO<sub>4</sub></li>
<li>1&nbsp;mM TCEP</li>
<li>100&nbsp;μM amiloride</li>
<li>1&nbsp;tablet per 100&nbsp;mL Pierce EDTA-free protease inhibitor tablets</li>
</ul>
<p>Membrane pellet was mixed thoroughly and homogenized with a Dounce homogenizer. Cell pellet was stirred for 1&nbsp; hour at 4&nbsp;°C. Solubilized membranes were centrifuged at 100,000&nbsp;xg for 45&nbsp;minutes at 4&nbsp;°C.</p>
<p>Next, 300&nbsp;μL GFP nanobody CNBr resin per gram of membrane was loaded into a gravity column and washed with 10&nbsp;CV <strong>HBS</strong></p>
<ul>
<li>20&nbsp;mM HEPES pH&nbsp;7.6</li>
<li>150&nbsp;mM NaCl)</li>
</ul>
<p>and 3&nbsp;CV <strong>wash A</strong></p>
<ul>
<li>0.1&nbsp;mg/mL GDN</li>
<li>20&nbsp;mM HEPES pH&nbsp;7.6</li>
<li>150&nbsp;mM NaCl</li>
<li>1&nbsp;mM TCEP</li>
<li>100&nbsp;nM phenamil</li>
</ul>
<p>Clarified lysate was flowed over this column under gravity. The column was washed with 6&nbsp;CV each of wash A, <strong>wash B</strong> (wash A with 2&nbsp;mM ATP, 2&nbsp;mM MgSO<sub>4</sub>), <strong>wash C</strong> (wash A with 25 U/mL ThermoFisher Universal Nuclease), and <strong>elution buffer</strong> (wash A with 5&nbsp;mM CaCl<sub>2</sub>). ENaC was eluted by flowing 1&nbsp;CV of elution buffer with 30&nbsp;μg/mL thrombin onto the column and collecting the supernatant. After an hour, this process was repeated. These elution steps were repeated for a total of four elutions of 1&nbsp;CV each.</p>
<p>The first three elutions were pooled and concentrated to 0.5&nbsp;mL. Fab was added at a one-to-one ratio with nominal protein concentration by nanodrop. Sample was incubated for 10&nbsp;minutes at room temperature, then centrifuged at 100,000&nbsp;xg for 20&nbsp;minutes at 4&nbsp;°C. The clarified sample was loaded onto a Superose 6 Increase 10/300 column (Cytiva) equilibrated in fresh wash A buffer. Fractions were assessed for purity via SDS-PAGE and FSEC before proceeding to grid preparation or other uses.</p>
</section>
</section>
<section id="enac-nanodisc-preparation" class="level2" data-number="A.3">
<h2 data-number="A.3" class="anchored" data-anchor-id="enac-nanodisc-preparation"><span class="header-section-number">A.3</span> ENaC nanodisc preparation</h2>
<p>To prepare nanodiscs, 15&nbsp;mg of a chloroform stock of POPC (Avanti) was dried down under an argon stream. The lipid film was left overnight in a vacuum desiccator to remove all chloroform. The lipids were then resuspended in 350 μL <strong>reconstitution buffer</strong></p>
<ul>
<li>20&nbsp;mM HEPES pH&nbsp;7.4</li>
<li>100&nbsp;mM KCl</li>
<li>1&nbsp;mM EDTA</li>
</ul>
<p>and sonicated until the solution was milky in appearance and no lipids remained adhered to the sides of the tube, typically 5–10 minutes. To this solution, 150 μL of 350 μM DM in reconstitution buffer was added. The resulting 500 μL suspension was sonicated in 5 minute bursts until completely clear. To this suspension, 1&nbsp;mL TBS was added to prepare a final lipid concentration of 10&nbsp;mg/mL.</p>
<p>Lipids, ENaC, and MSP2N2 were mixed at a final MSP:POPC:ENaC molar ratio of 1:31:0.04. The final concentration of MSP2N2 was kept at 100 μM by further concentrating ENaC or adding lipid reconstitution buffer with 35&nbsp;mM DM as necessary. This reconstitution mixture was incubated at room temperature for 1 hour. During this incubation, biobeads were washed once in 100% methanol, three times in water, and once in TBS. After 90 minutes, 215&nbsp;mg washed biobeads per 1&nbsp;mL reconstitution mixture were added. After 1 hour, the reconstitution mixture was moved to a fresh aliquot of equilibrated biobeads and left nutating overnight at room temperature, approximately 16 hours. The reconstitution mixture was pipetted off the biobeads, filtered with a 0.2 μm filter, and purified via SEC using a Superose 6 Increase 10/300 column in TBS. Typical yields were extremely low, between 5–10% of input ENaC. The protein assembly was stable with minimal aggregation at 4&nbsp;°C (assessed by cryoEM and SEC) for at least two weeks.</p>
</section>
<section id="western-blots" class="level2" data-number="A.4">
<h2 data-number="A.4" class="anchored" data-anchor-id="western-blots"><span class="header-section-number">A.4</span> Western Blots</h2>
<p>SDS-PAGE gels were blotted onto nitrocellulose membranes. Membranes were blocked in 50&nbsp;mg/mL milk in TBST for 1&nbsp;hour at room temperature. <!-- TODO: get lot number for gamma antibody --> Primary antibody (<strong>ENaC<sub>α</sub></strong>: Santa Cruz Biotechnology αENaC (H-95) sc-21012 lot no. L2812; <strong>ENaC<sub>γ</sub></strong>: abcam Anti-epithelial Sodium Channel gamma ab133430) was added to this same blocking buffer at 10&nbsp;μg/blot and incubated overnight at 4&nbsp;°C. Membranes were washed 3&nbsp;times with TBST for 5&nbsp;minutes before addition of secondary antibody (LI-COR IRDye 800 CW Goat anti-Mouse 925-32210 lot no. D21115-21) in TBST. Membranes were incubated at room temperature with secondary for 1&nbsp;hour, washed 3&nbsp;times with TBST and once with TBS, then imaged (LI-COR Odyssey DLx).</p>
</section>
<section id="cryoem-grid-preparation" class="level2" data-number="A.5">
<h2 data-number="A.5" class="anchored" data-anchor-id="cryoem-grid-preparation"><span class="header-section-number">A.5</span> cryoEM Grid Preparation</h2>
<p>Grid preparation different slightly for each dataset, but followed the same general procedure (<a href="#tbl-model-building">Table&nbsp;<span>A.1</span></a>). Purified GFP-cleaved ENaC was concentrated for grid preparation just before blotting and freezing. All grids were glow discharged at 15 mA for 60 seconds on a Pelco easiGlow before blotting. Grids were frozen using a Vitrobot Mark III (Thermo Fisher). Vitrobot parameters were set as follows: blot time 2, wait time 0, drain time 0, blot force 1, humidity 100%, temperature 12 °C. “Double-blotted” grids had a protein droplet loaded, manually blotted with torn Whatman filter paper, and a fresh droplet loaded onto the grid before Vitrobot blotting (with Vitrobot filter paper rings) and freezing. This double-blotting process typically took approximately five seconds from first drop to plunge-freeze. In all cases, each protein droplet was 3 μL.</p>
</section>
<section id="cryoem-data-collection" class="level2" data-number="A.6">
<h2 data-number="A.6" class="anchored" data-anchor-id="cryoem-data-collection"><span class="header-section-number">A.6</span> cryoEM Data Collection</h2>
<p>Datasets were collected using SerialEM by microscopy core staff at the facilities used (<a href="#tbl-model-building">Table&nbsp;<span>A.1</span></a>)<span class="citation" data-cites="mastronarde2003"><sup><a href="../references.html#ref-mastronarde2003" role="doc-biblioref">2</a></sup></span>. In all cases, multi-shot multi-hole regimes were used to maximize the number of movies recorded. Number of shots per hole and maximum beam shift were a function of ice quality and grid type, but typically a 3x3 pattern of holes was imaged 2–4 times per hole.</p>
</section>
<section id="cryoem-image-processing" class="level2" data-number="A.7">
<h2 data-number="A.7" class="anchored" data-anchor-id="cryoem-image-processing"><span class="header-section-number">A.7</span> cryoEM Image Processing</h2>
<p>Each dataset was processed slightly differently, but following the same general pipeline. First, movies were imported into cryoSPARC for pre-processing. Default parameters were used for both motion correction and CTF estimation. Next, particles were picked using the blob picker in cryoSPARC. Adjustments were made to the minimum and maximum radii to account for the number of Fabs present in each sample. Blob picks for several micrographs were manually inspected and adjusted. First, the NCC parameter was adjusted until few contaminant and carbon-edge picks remained. Next, in a low-defocus micrograph, the low power threshold was adjusted to remove empty ice picks. Finally, in a high-defocus micrograph, the high power threshold was adjusted to remove all carbon-edge and contaminant picks. These initial blob picks were cleaned by 2D classification, and the resulting particle sets used to generate templates for template picking. Template picked thresholds were adjusted as for blob picked particles.</p>
<p>The template picked particles were repeatedly 3D classified using heterogeneous refinement in cryoSPARC, providing one “good” class and a number of “junk” classes (typically noisy or bad <em>ab initio</em> models of the blob picked particles) until all particles produced reasonable ENaC classes. The final particle set was never 2D classified, since I find that 2D classification reduces the frequency of rare particle views. From this point, processing proceeded <em>ad hoc</em> to produce the final maps (Image Processing).</p>
<div id="fig-pixel-size-calibration" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/matmeth/ecd-pix-size/pixel-size.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="S2C2 pixel sizes seem miscalibrated. A: Scaling the nominal pixel size for human CKO/DEG dramatically increases the correlation with human CKO digitonin. B: Correlation coefficients for all maps. Maps were aligned prior to taking correlation coefficients. Abbreviated as follows: hCKO: human CKO; hCD-MF: human CKO/DEG monofab; mTrypsin: trypsin-cleaved mouse; hCKO/DEG: human CKO/DEG difab; hCKO-ND: human CKO nanodisc; mUncleaved: mouse uncleaved; mJanelia2: a second uncleaved mouse dataset collected at Janelia, but not presented here."><img src="figures/matmeth/ecd-pix-size/pixel-size.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;A.1: S<sup>2</sup>C<sup>2</sup> pixel sizes seem miscalibrated. <strong>A</strong>: Scaling the nominal pixel size for human CKO/DEG dramatically increases the correlation with human CKO digitonin. <strong>B</strong>: Correlation coefficients for all maps. Maps were aligned prior to taking correlation coefficients. Abbreviated as follows: hCKO: human CKO; hCD-MF: human CKO/DEG monofab; mTrypsin: trypsin-cleaved mouse; hCKO/DEG: human CKO/DEG difab; hCKO-ND: human CKO nanodisc; mUncleaved: mouse uncleaved; mJanelia2: a second uncleaved mouse dataset collected at Janelia, but not presented here.</figcaption>
</figure>
</div>
<p>During image processing, I began to suspect that the pixel sizes provided by S<sup>2</sup>C<sup>2</sup> are likely miscalibrated. First, other users reported pixel size calibration issues with S<sup>2</sup>C<sup>2</sup> microscopes around the time these datasets were collected (Patrick Mitchell, S<sup>2</sup>C<sup>2</sup> Director of Operations, personal communication). Second, scaling the CKO/DEG difab pixel size from the nominally-correct 0.43 to 0.414 (a change of approximately 3.6%) changes the map correlation value from 0.72 to 0.93 (<a href="#fig-pixel-size-calibration">Figure&nbsp;<span>A.1</span></a> A). Finally, a comparison of the correlation between seven recently-collected ENaC maps shows that S<sup>2</sup>C<sup>2</sup> is an outlier, with unscaled maps generated from data collected at PNCC and Janelia Farm correlating better with themselves and each other than with those collected at S<sup>2</sup>C<sup>2</sup> regardless of construct (<a href="#fig-pixel-size-calibration">Figure&nbsp;<span>A.1</span></a> B). I therefore decided to scale both S<sup>2</sup>C<sup>2</sup> maps to correlate best with the same condition and construct recorded at PNCC after processing was complete. I present the models built into the scaled maps in this work.</p>
</section>
<section id="model-building" class="level2" data-number="A.8">
<h2 data-number="A.8" class="anchored" data-anchor-id="model-building"><span class="header-section-number">A.8</span> Model Building</h2>
<p>The starting model for mouse ENaC was generated using a local installation of AlphaFold<span class="citation" data-cites="jumper2021"><sup><a href="../references.html#ref-jumper2021" role="doc-biblioref">3</a></sup></span>. The amino acid sequences for mENaC α (Uniprot ID Q61180), β (Uniprot ID Q9WU38), and γ (Uniprot ID Q9WU39) were joined using 10xGS linkers between α and β, and β and γ. This meta-sequence was then entered as the AlphaFold target. The linkers and large, unstructured, low-confidence loops were removed from the model before proceeding.</p>
<p>Before docking into experimental maps, starting models had Fabs and waters deleted to ease early model building steps. Initial models were rigid-body fitted into the experimental maps. Any loops in poor or missing regions of the map were deleted. The whole models were simulated with distance restraints using ISOLDE and ChimeraX<span class="citation" data-cites="croll2018 pettersen2021"><sup><a href="../references.html#ref-croll2018" role="doc-biblioref">4</a>,<a href="../references.html#ref-pettersen2021" role="doc-biblioref">5</a></sup></span>. Once models were roughly fit into the potential, strained distance restraints were released. Once the backbone appeared to fit reasonably well into the map, all distance restraints were released. Simulating one chain at a time, a pass over every residue in ENaC was performed, focusing on Ramachandran values and rotamers. Once all chains had been optimized in this manner, glycans were added using Coot’s carbohydrate module<span class="citation" data-cites="emsley2010"><sup><a href="../references.html#ref-emsley2010" role="doc-biblioref">6</a></sup></span>. Fabs were rigid-body fitted and simulated similarly to the initial models once these steps were complete.</p>
<p>Models were refined using Phenix real-space refinement<span class="citation" data-cites="liebschner2019"><sup><a href="../references.html#ref-liebschner2019" role="doc-biblioref">7</a></sup></span>. In some cases, the TMD map was used as-is for refinement. In other cases, the ECD and TMD map were combined using Phenix’s “Combine Focused Maps” job and the resulting map was used for refinement. In these cases, the lowest GSFSC resolution was provided to Phenix. Parameters were left as default, except rotamers were only refined when they had both poor map quality <em>and</em> were outliers (rather than <em>or</em>). I changed this setting because rotamer and clash statistics were significantly worse if it was left as the default. Model quality was assessed using MolProbity within Phenix, Q-scores calculated in ChimeraX, and 3DFSC as implemented in cryoSPARC (<a href="#tbl-model-building">Table&nbsp;<span>A.1</span></a> and <a href="validation.html"><span>Appendix&nbsp;C</span></a>)<span class="citation" data-cites="williams2018 pintilie2020 aiyer2021"><sup><a href="../references.html#ref-williams2018" role="doc-biblioref">8</a>–<a href="../references.html#ref-aiyer2021" role="doc-biblioref">10</a></sup></span>.</p>
</section>
<section id="constructs" class="level2" data-number="A.9">
<h2 data-number="A.9" class="anchored" data-anchor-id="constructs"><span class="header-section-number">A.9</span> Constructs</h2>

<!-- TODO: fix LaTeX table output -->
<div class="cell">
<div class="cell-output-display">
<div id="tbl-model-building" class="anchored">

<div id="qymtvjzehs" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#qymtvjzehs table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#qymtvjzehs thead, #qymtvjzehs tbody, #qymtvjzehs tfoot, #qymtvjzehs tr, #qymtvjzehs td, #qymtvjzehs th {
  border-style: none;
}

#qymtvjzehs p {
  margin: 0;
  padding: 0;
}

#qymtvjzehs .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 14px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#qymtvjzehs .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#qymtvjzehs .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#qymtvjzehs .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#qymtvjzehs .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qymtvjzehs .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qymtvjzehs .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qymtvjzehs .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#qymtvjzehs .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#qymtvjzehs .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#qymtvjzehs .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#qymtvjzehs .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#qymtvjzehs .gt_spanner_row {
  border-bottom-style: hidden;
}

#qymtvjzehs .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#qymtvjzehs .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#qymtvjzehs .gt_from_md > :first-child {
  margin-top: 0;
}

#qymtvjzehs .gt_from_md > :last-child {
  margin-bottom: 0;
}

#qymtvjzehs .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#qymtvjzehs .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#qymtvjzehs .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#qymtvjzehs .gt_row_group_first td {
  border-top-width: 2px;
}

#qymtvjzehs .gt_row_group_first th {
  border-top-width: 2px;
}

#qymtvjzehs .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qymtvjzehs .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#qymtvjzehs .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#qymtvjzehs .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qymtvjzehs .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qymtvjzehs .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#qymtvjzehs .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#qymtvjzehs .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#qymtvjzehs .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qymtvjzehs .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qymtvjzehs .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qymtvjzehs .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qymtvjzehs .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qymtvjzehs .gt_left {
  text-align: left;
}

#qymtvjzehs .gt_center {
  text-align: center;
}

#qymtvjzehs .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#qymtvjzehs .gt_font_normal {
  font-weight: normal;
}

#qymtvjzehs .gt_font_bold {
  font-weight: bold;
}

#qymtvjzehs .gt_font_italic {
  font-style: italic;
}

#qymtvjzehs .gt_super {
  font-size: 65%;
}

#qymtvjzehs .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#qymtvjzehs .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#qymtvjzehs .gt_indent_1 {
  text-indent: 5px;
}

#qymtvjzehs .gt_indent_2 {
  text-indent: 10px;
}

#qymtvjzehs .gt_indent_3 {
  text-indent: 15px;
}

#qymtvjzehs .gt_indent_4 {
  text-indent: 20px;
}

#qymtvjzehs .gt_indent_5 {
  text-indent: 25px;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false"><caption>Table&nbsp;A.1:  <p>Image processing and model building statistics.</p> </caption>
  <thead>
    
    <tr class="gt_col_headings gt_spanner_row">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="2" colspan="1" scope="col" id=""></th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="human CKO">
        <span class="gt_column_spanner">human CKO</span>
      </th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="human CKO/DEG">
        <span class="gt_column_spanner">human CKO/DEG</span>
      </th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="mouse">
        <span class="gt_column_spanner">mouse</span>
      </th>
    </tr>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Digitonin">Digitonin</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Nanodisc">Nanodisc</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Difab">Difab</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Monofab">Monofab</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Uncleaved">Uncleaved</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="Trypsin">Trypsin</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <th colspan="7" class="gt_group_heading" scope="colgroup" id="Construct Summary">Construct Summary</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_1" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>α mutations</p>
</div></th>
<td headers="Construct Summary stub_1_1 Digitonin" class="gt_row gt_center">C63A, R178A, R204A</td>
<td headers="Construct Summary stub_1_1 Nanodisc" class="gt_row gt_center">C63A, R178A, R204A</td>
<td headers="Construct Summary stub_1_1 Difab" class="gt_row gt_center">C63A, R178A, R204A, T240W</td>
<td headers="Construct Summary stub_1_1 Monofab" class="gt_row gt_center">C63A, R178A, R204A, T240W</td>
<td headers="Construct Summary stub_1_1 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_1 Trypsin" class="gt_row gt_center">—</td></tr>
    <tr><th id="stub_1_2" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>β mutations</p>
</div></th>
<td headers="Construct Summary stub_1_2 Digitonin" class="gt_row gt_center">C30A</td>
<td headers="Construct Summary stub_1_2 Nanodisc" class="gt_row gt_center">C30A</td>
<td headers="Construct Summary stub_1_2 Difab" class="gt_row gt_center">C30A, S520K</td>
<td headers="Construct Summary stub_1_2 Monofab" class="gt_row gt_center">C30A, S520K</td>
<td headers="Construct Summary stub_1_2 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_2 Trypsin" class="gt_row gt_center">—</td></tr>
    <tr><th id="stub_1_3" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>γ mutations</p>
</div></th>
<td headers="Construct Summary stub_1_3 Digitonin" class="gt_row gt_center">C33A, C41A, R138A</td>
<td headers="Construct Summary stub_1_3 Nanodisc" class="gt_row gt_center">C33A, C41A, R138A</td>
<td headers="Construct Summary stub_1_3 Difab" class="gt_row gt_center">C33A, C41A, R138A</td>
<td headers="Construct Summary stub_1_3 Monofab" class="gt_row gt_center">C33A, C41A, R138A</td>
<td headers="Construct Summary stub_1_3 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_3 Trypsin" class="gt_row gt_center">—</td></tr>
    <tr><th id="stub_1_4" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Other treatments</p>
</div></th>
<td headers="Construct Summary stub_1_4 Digitonin" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_4 Nanodisc" class="gt_row gt_center">POPC nanodisc</td>
<td headers="Construct Summary stub_1_4 Difab" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_4 Monofab" class="gt_row gt_center">1 mM C8 PIP₂</td>
<td headers="Construct Summary stub_1_4 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Construct Summary stub_1_4 Trypsin" class="gt_row gt_center">10 μg/mL trypsin for 5 minutes</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="7" class="gt_group_heading" scope="colgroup" id="Grid Preparation">Grid Preparation</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_5" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>ENaC (μM)</p>
</div></th>
<td headers="Grid Preparation stub_1_5 Digitonin" class="gt_row gt_center">19.8</td>
<td headers="Grid Preparation stub_1_5 Nanodisc" class="gt_row gt_center">2</td>
<td headers="Grid Preparation stub_1_5 Difab" class="gt_row gt_center">21.3</td>
<td headers="Grid Preparation stub_1_5 Monofab" class="gt_row gt_center">24</td>
<td headers="Grid Preparation stub_1_5 Uncleaved" class="gt_row gt_center">9.6</td>
<td headers="Grid Preparation stub_1_5 Trypsin" class="gt_row gt_center">9.6</td></tr>
    <tr><th id="stub_1_6" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Salt</p>
</div></th>
<td headers="Grid Preparation stub_1_6 Digitonin" class="gt_row gt_center">NaCl</td>
<td headers="Grid Preparation stub_1_6 Nanodisc" class="gt_row gt_center">KCl</td>
<td headers="Grid Preparation stub_1_6 Difab" class="gt_row gt_center">KCl</td>
<td headers="Grid Preparation stub_1_6 Monofab" class="gt_row gt_center">KCl</td>
<td headers="Grid Preparation stub_1_6 Uncleaved" class="gt_row gt_center">KCl</td>
<td headers="Grid Preparation stub_1_6 Trypsin" class="gt_row gt_center">KCl</td></tr>
    <tr><th id="stub_1_7" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Acid Shock</p>
</div></th>
<td headers="Grid Preparation stub_1_7 Digitonin" class="gt_row gt_center">Yes</td>
<td headers="Grid Preparation stub_1_7 Nanodisc" class="gt_row gt_center">No</td>
<td headers="Grid Preparation stub_1_7 Difab" class="gt_row gt_center">Yes</td>
<td headers="Grid Preparation stub_1_7 Monofab" class="gt_row gt_center">No</td>
<td headers="Grid Preparation stub_1_7 Uncleaved" class="gt_row gt_center">No</td>
<td headers="Grid Preparation stub_1_7 Trypsin" class="gt_row gt_center">No</td></tr>
    <tr><th id="stub_1_8" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Fabs</p>
</div></th>
<td headers="Grid Preparation stub_1_8 Digitonin" class="gt_row gt_center">α and β</td>
<td headers="Grid Preparation stub_1_8 Nanodisc" class="gt_row gt_center">—</td>
<td headers="Grid Preparation stub_1_8 Difab" class="gt_row gt_center">α and β</td>
<td headers="Grid Preparation stub_1_8 Monofab" class="gt_row gt_center">α</td>
<td headers="Grid Preparation stub_1_8 Uncleaved" class="gt_row gt_center">α</td>
<td headers="Grid Preparation stub_1_8 Trypsin" class="gt_row gt_center">α</td></tr>
    <tr><th id="stub_1_9" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Grid Type<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>1</sup></span></p></div></th>
<td headers="Grid Preparation stub_1_9 Digitonin" class="gt_row gt_center">2/1 200</td>
<td headers="Grid Preparation stub_1_9 Nanodisc" class="gt_row gt_center">2/2 200 + 2 nm carbon</td>
<td headers="Grid Preparation stub_1_9 Difab" class="gt_row gt_center">2/1 200</td>
<td headers="Grid Preparation stub_1_9 Monofab" class="gt_row gt_center">2/1 200</td>
<td headers="Grid Preparation stub_1_9 Uncleaved" class="gt_row gt_center">2/2 200</td>
<td headers="Grid Preparation stub_1_9 Trypsin" class="gt_row gt_center">2/1 200</td></tr>
    <tr><th id="stub_1_10" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>FOM (μM)</p>
</div></th>
<td headers="Grid Preparation stub_1_10 Digitonin" class="gt_row gt_center">2</td>
<td headers="Grid Preparation stub_1_10 Nanodisc" class="gt_row gt_center">10</td>
<td headers="Grid Preparation stub_1_10 Difab" class="gt_row gt_center">10</td>
<td headers="Grid Preparation stub_1_10 Monofab" class="gt_row gt_center">2</td>
<td headers="Grid Preparation stub_1_10 Uncleaved" class="gt_row gt_center">100</td>
<td headers="Grid Preparation stub_1_10 Trypsin" class="gt_row gt_center">2</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="7" class="gt_group_heading" scope="colgroup" id="Microscopy">Microscopy</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_11" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Facility</p>
</div></th>
<td headers="Microscopy stub_1_11 Digitonin" class="gt_row gt_center">PNCC</td>
<td headers="Microscopy stub_1_11 Nanodisc" class="gt_row gt_center">S2C2</td>
<td headers="Microscopy stub_1_11 Difab" class="gt_row gt_center">S2C2</td>
<td headers="Microscopy stub_1_11 Monofab" class="gt_row gt_center">PNCC</td>
<td headers="Microscopy stub_1_11 Uncleaved" class="gt_row gt_center">Janelia Farm</td>
<td headers="Microscopy stub_1_11 Trypsin" class="gt_row gt_center">Janelia Farm</td></tr>
    <tr><th id="stub_1_12" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>No. Movies</p>
</div></th>
<td headers="Microscopy stub_1_12 Digitonin" class="gt_row gt_center">13,689</td>
<td headers="Microscopy stub_1_12 Nanodisc" class="gt_row gt_center">9,882</td>
<td headers="Microscopy stub_1_12 Difab" class="gt_row gt_center">5,126</td>
<td headers="Microscopy stub_1_12 Monofab" class="gt_row gt_center">5,832</td>
<td headers="Microscopy stub_1_12 Uncleaved" class="gt_row gt_center">11,334</td>
<td headers="Microscopy stub_1_12 Trypsin" class="gt_row gt_center">6,960</td></tr>
    <tr><th id="stub_1_13" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Frames<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>2</sup></span></p></div></th>
<td headers="Microscopy stub_1_13 Digitonin" class="gt_row gt_center">50</td>
<td headers="Microscopy stub_1_13 Nanodisc" class="gt_row gt_center">65</td>
<td headers="Microscopy stub_1_13 Difab" class="gt_row gt_center">50</td>
<td headers="Microscopy stub_1_13 Monofab" class="gt_row gt_center">50</td>
<td headers="Microscopy stub_1_13 Uncleaved" class="gt_row gt_center">65</td>
<td headers="Microscopy stub_1_13 Trypsin" class="gt_row gt_center">65</td></tr>
    <tr><th id="stub_1_14" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Pixel Size (Å/pix)<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>3,4,5</sup></span></p></div></th>
<td headers="Microscopy stub_1_14 Digitonin" class="gt_row gt_center">0.8015</td>
<td headers="Microscopy stub_1_14 Nanodisc" class="gt_row gt_center">0.86</td>
<td headers="Microscopy stub_1_14 Difab" class="gt_row gt_center">0.86</td>
<td headers="Microscopy stub_1_14 Monofab" class="gt_row gt_center">0.830</td>
<td headers="Microscopy stub_1_14 Uncleaved" class="gt_row gt_center">0.844</td>
<td headers="Microscopy stub_1_14 Trypsin" class="gt_row gt_center">0.788</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="7" class="gt_group_heading" scope="colgroup" id="Processing">Processing</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_15" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>No. particles</p>
</div></th>
<td headers="Processing stub_1_15 Digitonin" class="gt_row gt_center">612,591</td>
<td headers="Processing stub_1_15 Nanodisc" class="gt_row gt_center">237,730</td>
<td headers="Processing stub_1_15 Difab" class="gt_row gt_center">257,341</td>
<td headers="Processing stub_1_15 Monofab" class="gt_row gt_center">373,495</td>
<td headers="Processing stub_1_15 Uncleaved" class="gt_row gt_center">499,081</td>
<td headers="Processing stub_1_15 Trypsin" class="gt_row gt_center">264,228</td></tr>
    <tr><th id="stub_1_16" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>ECD resolution (Å)<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>6,7</sup></span></p></div></th>
<td headers="Processing stub_1_16 Digitonin" class="gt_row gt_center">2.34</td>
<td headers="Processing stub_1_16 Nanodisc" class="gt_row gt_center">3.06</td>
<td headers="Processing stub_1_16 Difab" class="gt_row gt_center">2.94</td>
<td headers="Processing stub_1_16 Monofab" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_16 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_16 Trypsin" class="gt_row gt_center">3.03</td></tr>
    <tr><th id="stub_1_17" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Whole protein resolution (Å)<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>6,7</sup></span></p></div></th>
<td headers="Processing stub_1_17 Digitonin" class="gt_row gt_center">2.89</td>
<td headers="Processing stub_1_17 Nanodisc" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_17 Difab" class="gt_row gt_center">3.26</td>
<td headers="Processing stub_1_17 Monofab" class="gt_row gt_center">3.04</td>
<td headers="Processing stub_1_17 Uncleaved" class="gt_row gt_center">3.24</td>
<td headers="Processing stub_1_17 Trypsin" class="gt_row gt_center">3.30</td></tr>
    <tr><th id="stub_1_18" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>ECD sharpening factor<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>6</sup></span></p></div></th>
<td headers="Processing stub_1_18 Digitonin" class="gt_row gt_center">-56.6</td>
<td headers="Processing stub_1_18 Nanodisc" class="gt_row gt_center">-77.1</td>
<td headers="Processing stub_1_18 Difab" class="gt_row gt_center">-88.9</td>
<td headers="Processing stub_1_18 Monofab" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_18 Uncleaved" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_18 Trypsin" class="gt_row gt_center">-66.7</td></tr>
    <tr><th id="stub_1_19" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Whole protein sharpening factor<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>6</sup></span></p></div></th>
<td headers="Processing stub_1_19 Digitonin" class="gt_row gt_center">-66.0</td>
<td headers="Processing stub_1_19 Nanodisc" class="gt_row gt_center">—</td>
<td headers="Processing stub_1_19 Difab" class="gt_row gt_center">-110.4</td>
<td headers="Processing stub_1_19 Monofab" class="gt_row gt_center">-104.4</td>
<td headers="Processing stub_1_19 Uncleaved" class="gt_row gt_center">-96.4</td>
<td headers="Processing stub_1_19 Trypsin" class="gt_row gt_center">-67.4</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="7" class="gt_group_heading" scope="colgroup" id="Model Building">Model Building</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_20" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Starting model</p>
</div></th>
<td headers="Model Building stub_1_20 Digitonin" class="gt_row gt_center">6WTH</td>
<td headers="Model Building stub_1_20 Nanodisc" class="gt_row gt_center">CKO digitonin</td>
<td headers="Model Building stub_1_20 Difab" class="gt_row gt_center">6WTH</td>
<td headers="Model Building stub_1_20 Monofab" class="gt_row gt_center">CKO/DEG difab</td>
<td headers="Model Building stub_1_20 Uncleaved" class="gt_row gt_center">AlphaFold</td>
<td headers="Model Building stub_1_20 Trypsin" class="gt_row gt_center">mouse uncleaved</td></tr>
    <tr><th id="stub_1_21" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>No. non-H atoms</p>
</div></th>
<td headers="Model Building stub_1_21 Digitonin" class="gt_row gt_center">12,416</td>
<td headers="Model Building stub_1_21 Nanodisc" class="gt_row gt_center">9,971</td>
<td headers="Model Building stub_1_21 Difab" class="gt_row gt_center">12,807</td>
<td headers="Model Building stub_1_21 Monofab" class="gt_row gt_center">11,588</td>
<td headers="Model Building stub_1_21 Uncleaved" class="gt_row gt_center">11,466</td>
<td headers="Model Building stub_1_21 Trypsin" class="gt_row gt_center">11,617</td></tr>
    <tr><th id="stub_1_22" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>No. residues</p>
</div></th>
<td headers="Model Building stub_1_22 Digitonin" class="gt_row gt_center">1,628</td>
<td headers="Model Building stub_1_22 Nanodisc" class="gt_row gt_center">1,174</td>
<td headers="Model Building stub_1_22 Difab" class="gt_row gt_center">1,714</td>
<td headers="Model Building stub_1_22 Monofab" class="gt_row gt_center">1,476</td>
<td headers="Model Building stub_1_22 Uncleaved" class="gt_row gt_center">1,506</td>
<td headers="Model Building stub_1_22 Trypsin" class="gt_row gt_center">1,540</td></tr>
    <tr><th id="stub_1_23" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>No. ligands<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>8</sup></span></p></div></th>
<td headers="Model Building stub_1_23 Digitonin" class="gt_row gt_center">49</td>
<td headers="Model Building stub_1_23 Nanodisc" class="gt_row gt_center">38</td>
<td headers="Model Building stub_1_23 Difab" class="gt_row gt_center">35</td>
<td headers="Model Building stub_1_23 Monofab" class="gt_row gt_center">43</td>
<td headers="Model Building stub_1_23 Uncleaved" class="gt_row gt_center">26</td>
<td headers="Model Building stub_1_23 Trypsin" class="gt_row gt_center">29</td></tr>
    <tr><th id="stub_1_24" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Bond length RMSD</p>
</div></th>
<td headers="Model Building stub_1_24 Digitonin" class="gt_row gt_center">0.006</td>
<td headers="Model Building stub_1_24 Nanodisc" class="gt_row gt_center">0.003</td>
<td headers="Model Building stub_1_24 Difab" class="gt_row gt_center">0.003</td>
<td headers="Model Building stub_1_24 Monofab" class="gt_row gt_center">0.004</td>
<td headers="Model Building stub_1_24 Uncleaved" class="gt_row gt_center">0.003</td>
<td headers="Model Building stub_1_24 Trypsin" class="gt_row gt_center">0.003</td></tr>
    <tr><th id="stub_1_25" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Bond angle RMSD</p>
</div></th>
<td headers="Model Building stub_1_25 Digitonin" class="gt_row gt_center">0.709</td>
<td headers="Model Building stub_1_25 Nanodisc" class="gt_row gt_center">0.505</td>
<td headers="Model Building stub_1_25 Difab" class="gt_row gt_center">0.519</td>
<td headers="Model Building stub_1_25 Monofab" class="gt_row gt_center">0.499</td>
<td headers="Model Building stub_1_25 Uncleaved" class="gt_row gt_center">0.438</td>
<td headers="Model Building stub_1_25 Trypsin" class="gt_row gt_center">0.446</td></tr>
    <tr><th id="stub_1_26" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Molprobity score</p>
</div></th>
<td headers="Model Building stub_1_26 Digitonin" class="gt_row gt_center">1.49</td>
<td headers="Model Building stub_1_26 Nanodisc" class="gt_row gt_center">1.11</td>
<td headers="Model Building stub_1_26 Difab" class="gt_row gt_center">1.10</td>
<td headers="Model Building stub_1_26 Monofab" class="gt_row gt_center">1.27</td>
<td headers="Model Building stub_1_26 Uncleaved" class="gt_row gt_center">1.48</td>
<td headers="Model Building stub_1_26 Trypsin" class="gt_row gt_center">1.44</td></tr>
    <tr><th id="stub_1_27" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Clash score</p>
</div></th>
<td headers="Model Building stub_1_27 Digitonin" class="gt_row gt_center">4.61</td>
<td headers="Model Building stub_1_27 Nanodisc" class="gt_row gt_center">3.14</td>
<td headers="Model Building stub_1_27 Difab" class="gt_row gt_center">3.06</td>
<td headers="Model Building stub_1_27 Monofab" class="gt_row gt_center">5.03</td>
<td headers="Model Building stub_1_27 Uncleaved" class="gt_row gt_center">6.44</td>
<td headers="Model Building stub_1_27 Trypsin" class="gt_row gt_center">5.70</td></tr>
    <tr><th id="stub_1_28" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Rama. outliers (%)</p>
</div></th>
<td headers="Model Building stub_1_28 Digitonin" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_28 Nanodisc" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_28 Difab" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_28 Monofab" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_28 Uncleaved" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_28 Trypsin" class="gt_row gt_center">0.00</td></tr>
    <tr><th id="stub_1_29" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Rama. allowed (%)</p>
</div></th>
<td headers="Model Building stub_1_29 Digitonin" class="gt_row gt_center">2.00</td>
<td headers="Model Building stub_1_29 Nanodisc" class="gt_row gt_center">1.57</td>
<td headers="Model Building stub_1_29 Difab" class="gt_row gt_center">1.26</td>
<td headers="Model Building stub_1_29 Monofab" class="gt_row gt_center">1.82</td>
<td headers="Model Building stub_1_29 Uncleaved" class="gt_row gt_center">2.65</td>
<td headers="Model Building stub_1_29 Trypsin" class="gt_row gt_center">2.67</td></tr>
    <tr><th id="stub_1_30" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Rama. favored (%)</p>
</div></th>
<td headers="Model Building stub_1_30 Digitonin" class="gt_row gt_center">98.00</td>
<td headers="Model Building stub_1_30 Nanodisc" class="gt_row gt_center">98.43</td>
<td headers="Model Building stub_1_30 Difab" class="gt_row gt_center">98.74</td>
<td headers="Model Building stub_1_30 Monofab" class="gt_row gt_center">98.18</td>
<td headers="Model Building stub_1_30 Uncleaved" class="gt_row gt_center">97.35</td>
<td headers="Model Building stub_1_30 Trypsin" class="gt_row gt_center">97.33</td></tr>
    <tr><th id="stub_1_31" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Rotamer outliers</p>
</div></th>
<td headers="Model Building stub_1_31 Digitonin" class="gt_row gt_center">2.15</td>
<td headers="Model Building stub_1_31 Nanodisc" class="gt_row gt_center">0.09</td>
<td headers="Model Building stub_1_31 Difab" class="gt_row gt_center">0.72</td>
<td headers="Model Building stub_1_31 Monofab" class="gt_row gt_center">0.28</td>
<td headers="Model Building stub_1_31 Uncleaved" class="gt_row gt_center">0.09</td>
<td headers="Model Building stub_1_31 Trypsin" class="gt_row gt_center">0.00</td></tr>
    <tr><th id="stub_1_32" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Cβ outliers</p>
</div></th>
<td headers="Model Building stub_1_32 Digitonin" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_32 Nanodisc" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_32 Difab" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_32 Monofab" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_32 Uncleaved" class="gt_row gt_center">0.00</td>
<td headers="Model Building stub_1_32 Trypsin" class="gt_row gt_center">0.00</td></tr>
    <tr><th id="stub_1_33" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>CaBLAM outliers</p>
</div></th>
<td headers="Model Building stub_1_33 Digitonin" class="gt_row gt_center">0.81</td>
<td headers="Model Building stub_1_33 Nanodisc" class="gt_row gt_center">0.44</td>
<td headers="Model Building stub_1_33 Difab" class="gt_row gt_center">0.40</td>
<td headers="Model Building stub_1_33 Monofab" class="gt_row gt_center">1.15</td>
<td headers="Model Building stub_1_33 Uncleaved" class="gt_row gt_center">1.52</td>
<td headers="Model Building stub_1_33 Trypsin" class="gt_row gt_center">0.86</td></tr>
    <tr><th id="stub_1_34" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Resolution (0.143 FSC)</p>
</div></th>
<td headers="Model Building stub_1_34 Digitonin" class="gt_row gt_center">2.3</td>
<td headers="Model Building stub_1_34 Nanodisc" class="gt_row gt_center">2.9</td>
<td headers="Model Building stub_1_34 Difab" class="gt_row gt_center">2.8</td>
<td headers="Model Building stub_1_34 Monofab" class="gt_row gt_center">3.1</td>
<td headers="Model Building stub_1_34 Uncleaved" class="gt_row gt_center">3.2</td>
<td headers="Model Building stub_1_34 Trypsin" class="gt_row gt_center">3.0</td></tr>
    <tr><th id="stub_1_35" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>CC (mask)</p>
</div></th>
<td headers="Model Building stub_1_35 Digitonin" class="gt_row gt_center">0.85</td>
<td headers="Model Building stub_1_35 Nanodisc" class="gt_row gt_center">0.84</td>
<td headers="Model Building stub_1_35 Difab" class="gt_row gt_center">0.83</td>
<td headers="Model Building stub_1_35 Monofab" class="gt_row gt_center">0.82</td>
<td headers="Model Building stub_1_35 Uncleaved" class="gt_row gt_center">0.83</td>
<td headers="Model Building stub_1_35 Trypsin" class="gt_row gt_center">0.84</td></tr>
    <tr><th id="stub_1_36" scope="row" class="gt_row gt_center gt_stub" style="border-right-width: 2; border-right-style: solid; border-right-color: #CCCCCC;"><div class="gt_from_md"><p>Q-score<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>9</sup></span></p></div></th>
<td headers="Model Building stub_1_36 Digitonin" class="gt_row gt_center">0.80</td>
<td headers="Model Building stub_1_36 Nanodisc" class="gt_row gt_center">0.71</td>
<td headers="Model Building stub_1_36 Difab" class="gt_row gt_center">0.71</td>
<td headers="Model Building stub_1_36 Monofab" class="gt_row gt_center">0.71</td>
<td headers="Model Building stub_1_36 Uncleaved" class="gt_row gt_center">0.68</td>
<td headers="Model Building stub_1_36 Trypsin" class="gt_row gt_center">0.72</td></tr>
  </tbody>
  
  <tfoot class="gt_footnotes">
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>1</sup></span> All grids are Quantifoil grids with gold mesh</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>2</sup></span> Electron flux is 1 e⁻/Å²/frame</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>3</sup></span> All datsets collected in super-resolution mode and Fourier-cropped back to this physical size.</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>4</sup></span> Resolutions here are nominal resolutions given by the microscopy facility. I have resion to believe the S²C² pixel size is incorrect, and built the models into a corrected pixel size (see Materials and Methods).</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>5</sup></span> These values were used for processing and are listed to the precision given by the microscopy facility at the time of collection.</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>6</sup></span> A dash indicates that a focused map for this region was not used in model building</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>7</sup></span> GSFSC resolution</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>8</sup></span> NAG, MAN, and BMA. UNK is counted as a residue instead.</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="7"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;"><sup>9</sup></span> Residues modeled as UNK excluded from Q-score</td>
    </tr>
  </tfoot>
</table>
</div>
</div>
</div>
</div>
<p>The four constructs I present here can broadly be split into those I expect to be open (CKO/DEG and mouse trypsin) and those I expect to be closed (CKO, mouse) (<a href="#tbl-model-building">Table&nbsp;<span>A.1</span></a>). All human channels in this work have mutant furin sites (αR178A R204A and γR138A). Additionally, all human channels are “cysteine knock-out” (CKO) mutants, with the following mutations: αC63A, βC30A, and γC33A C41A. Some (but not all) cysteines in the ENaC TMD are known to be palmitoylated<span class="citation" data-cites="mukherjee14 mueller10 mueller12"><sup><a href="../references.html#ref-mukherjee14" role="doc-biblioref">11</a>–<a href="../references.html#ref-mueller12" role="doc-biblioref">13</a></sup></span>. CKO channels express at approximately eight times the yield of wild-type-like channels, with similar electrophysiological traits (<a href="#fig-ephys">Figure&nbsp;<span>A.2</span></a>). Mouse channels presented in this work have no mutations from the consensus gene. I expect mouse and CKO channels to be closed.</p>
<div id="fig-ephys" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/matmeth/ephys/491-mean-iv.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="I/V curves for human CKO and CKO/DEG channels. Values are normalized per-cell to the inward current at -80 mV before trypsin treatment. Cells were treated with trypsin for five minutes before being re-recorded. Each point is the average of three cells. Error bars represent +/- one standard deviation."><img src="figures/matmeth/ephys/491-mean-iv.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;A.2: I/V curves for human CKO and CKO/DEG channels. Values are normalized per-cell to the inward current at -80 mV before trypsin treatment. Cells were treated with trypsin for five minutes before being re-recorded. Each point is the average of three cells. Error bars represent +/- one standard deviation.</figcaption>
</figure>
</div>
<p>DEG channels have two mutations: βS520K and αT240W. The former mutation at what has been named the “DEG site” holds the channel in an open state regardless of proteolytic cleavage state<span class="citation" data-cites="snyder2000"><sup><a href="../references.html#ref-snyder2000" role="doc-biblioref">14</a></sup></span>. The DEG site sits in the TMD just above the predicted amiloride-binding site. The latter mutation in α lies in the GRIP domain and also increases the P<sub>O</sub> of uncleaved ENaC<span class="citation" data-cites="kashlan2010"><sup><a href="../references.html#ref-kashlan2010" role="doc-biblioref">15</a></sup></span>.</p>
</section>
<section id="whole-cell-patch-clamp" class="level2" data-number="A.10">
<h2 data-number="A.10" class="anchored" data-anchor-id="whole-cell-patch-clamp"><span class="header-section-number">A.10</span> Whole cell patch clamp</h2>
<p>Human embryonic kidney cells (HEK293S/17) were grown in suspension in Freestyle medium with 10% FBS to a density of 1<span class="math inline">\(\times 10^6\)</span> cells/mL, at which point they were infected with virus carrying the appropriate ENaC genes. Cells were incubated at 37&nbsp;°C overnight with 1 μM amiloride. Cells were then pipetted over a dish of DMEM with 10% FBS with several small coverslips and allowed to adhere at 37&nbsp;°C while the electrophysiology rig was prepared, approximately 2 hours. Pipettes were pulled to 2–3 MΩ resistance and filled with filtered <strong>internal solution</strong></p>
<ul>
<li>150&nbsp;mM KCl</li>
<li>2&nbsp;mM MgCl<sub>2</sub></li>
<li>5&nbsp;mM EGTA</li>
<li>10&nbsp;mM HEPES pH&nbsp;7.4</li>
</ul>
<p>Working one coverslip at a time, cells were moved from DMEM at 37&nbsp;°C to a dish on the rig (at room temperature) containing <strong>K<sup>+</sup> external solution</strong></p>
<ul>
<li>150&nbsp;mM KCl</li>
<li>2&nbsp;mM MgCl<sub>2</sub></li>
<li>2&nbsp;mM CaCl<sub>2</sub></li>
<li>and 10&nbsp;mM HEPES pH&nbsp;7.4</li>
</ul>
<p>Also prepared were solutions of Na<sup>+</sup> and Li<sup>+</sup>, the same as the K<sup>+</sup> solution but with NaCl or LiCl replacing the KCl, respectively. Each salt solution also had a counterpart with 100 μM amiloride added. Finally, a <strong>trypsin solution</strong> (KCl external solution with 5 μg/mL trypsin) was prepared.</p>
<p>Amiloride sensitive current was determined by placing a cell in the amiloride external solution for 1 second, the amiloride-free solution for 2 seconds, and returning to the amiloride solution for 1 second. Average current during no-amiloride traces was subtracted to set the baseline to 0 A, and the maximum current during the 2 second no-amiloride duration taken as the amiloride-sensitive current. The K<sup>+</sup> recordings show a current spike when returning to the amiloride-containing solution, which I exclude from the calculations. This process was performed once for each holding potential between -80 and +80 mV, stepping by 20 mV. Then, this entire voltage sweep was repeated for each of the two remaining salts. The cell was then exposed to the trypsin solution for 5 minutes, and the entire set of voltage and salt sweeps repeated for the post-trypsin measurements. Thus, the full combination of voltage, salt, and proteolytic cleavage states was recorded on each cell. Once a dish had been exposed to the trypsin solution, the dish was replaced with a fresh dish and coverslip to prevent early cleavage of unmeasured cells.</p>


<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-kawate06" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Kawate, T., and Gouaux, E. (2006/04/01/). Fluorescence-<span>Detection Size-Exclusion Chromatography</span> for <span>Precrystallization Screening</span> of <span>Integral Membrane Proteins</span>. Structure <em>14</em>, 673–681. <a href="https://doi.org/10.1016/j.str.2006.01.013">10.1016/j.str.2006.01.013</a>.</div>
</div>
<div id="ref-mastronarde2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Mastronarde, D.N. (2003). <span>SerialEM</span>: <span>A Program</span> for <span>Automated Tilt Series Acquisition</span> on <span>Tecnai Microscopes Using Prediction</span> of <span>Specimen Position</span>. Microscopy and Microanalysis <em>9</em>, 1182–1183. <a href="https://doi.org/10.1017/S1431927603445911">10.1017/S1431927603445911</a>.</div>
</div>
<div id="ref-jumper2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., Tunyasuvunakool, K., Bates, R., Žídek, A., Potapenko, A., et al. (2021). Highly accurate protein structure prediction with <span>AlphaFold</span>. Nature <em>596</em>, 583–589. <a href="https://doi.org/10.1038/s41586-021-03819-2">10.1038/s41586-021-03819-2</a>.</div>
</div>
<div id="ref-croll2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Croll, T. (2018). <a href="https://doi.org/10.1107/S2059798318002425"><span>ISOLDE</span>: A physically realistic environment for model building into low-resolution electron-density maps</a>. Acta Crystallographica Section D <em>74</em>, 519–530.</div>
</div>
<div id="ref-pettersen2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Pettersen, E.F., Goddard, T.D., Huang, C.C., Meng, E.C., Couch, G.S., Croll, T.I., Morris, J.H., and Ferrin, T.E. (2021). <span>UCSF ChimeraX</span>: <span>Structure</span> visualization for researchers, educators, and developers. Protein Sci <em>30</em>, 70–82. <a href="https://doi.org/10.1002/pro.3943">10.1002/pro.3943</a>.</div>
</div>
<div id="ref-emsley2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). <a href="https://doi.org/10.1107/S0907444910007493">Features and development of <span>Coot</span></a>. Acta Crystallographica Section D <em>66</em>, 486–501.</div>
</div>
<div id="ref-liebschner2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Liebschner, D., Afonine, P.V., Baker, M.L., Bunkoczi, G., Chen, V.B., Croll, T.I., Hintze, B., Hung, L.-W., Jain, S., McCoy, A.J., et al. (2019). <a href="https://doi.org/10.1107/S2059798319011471">Macromolecular structure determination using <span class="nocase">X-rays</span>, neutrons and electrons: Recent developments in <span>Phenix</span></a>. Acta Crystallographica Section D <em>75</em>, 861–877.</div>
</div>
<div id="ref-williams2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Williams, C.J., Headd, J.J., Moriarty, N.W., Prisant, M.G., Videau, L.L., Deis, L.N., Verma, V., Keedy, D.A., Hintze, B.J., Chen, V.B., et al. (2018). <span>MolProbity</span>: <span>More</span> and better reference data for improved all-atom structure validation. Protein Science <em>27</em>, 293–315. <a href="https://doi.org/10.1002/pro.3330">10.1002/pro.3330</a>.</div>
</div>
<div id="ref-pintilie2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Pintilie, G., Zhang, K., Su, Z., Li, S., Schmid, M.F., and Chiu, W. (2020). Measurement of atom resolvability in cryo-<span>EM</span> maps with <span class="nocase">Q-scores</span>. Nature Methods <em>17</em>, 328–334. <a href="https://doi.org/10.1038/s41592-020-0731-1">10.1038/s41592-020-0731-1</a>.</div>
</div>
<div id="ref-aiyer2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Aiyer, S., Zhang, C., Baldwin, P.R., and Lyumkis, D. (2021). Evaluating <span>Local</span> and <span>Directional Resolution</span> of <span class="nocase">Cryo-EMCryo-electron</span> microscopy (<span>Cryo-EM</span>) <span>Density Maps</span>. In <span class="nocase">cryoEM</span>: <span>Methods</span> and <span>Protocols</span>, T. Gonen and B. L. Nannenga, eds. (<span>Springer US</span>), pp. 161–187. <a href="https://doi.org/10.1007/978-1-0716-0966-8_8">10.1007/978-1-0716-0966-8_8</a>.</div>
</div>
<div id="ref-mukherjee14" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Mukherjee, A., Mueller, G.M., Kinlough, C.L., Sheng, N., Wang, Z., Mustafa, S.A., Kashlan, O.B., Kleyman, T.R., and Hughey, R.P. (2014/05/16/). Cysteine <span>Palmitoylation</span> of the γ <span>Subunit Has</span> a <span>Dominant Role</span> in <span>Modulating Activity</span> of the <span>Epithelial Sodium Channel</span>*. J. Biol. Chem. <em>289</em>, 14351–14359. <a href="https://doi.org/10.1074/jbc.M113.526020">10.1074/jbc.M113.526020</a>.</div>
</div>
<div id="ref-mueller10" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Mueller, G.M., Maarouf, A.B., Kinlough, C.L., Sheng, N., Kashlan, O.B., Okumura, S., Luthy, S., Kleyman, T.R., and Hughey, R.P. (2010). Cys <span>Palmitoylation</span> of the <span>Beta Subunit Modulates Gating</span> of the <span>Epithelial Sodium Channel</span>. J. Biol. Chem. <em>285</em>, 30453–30462. <a href="https://doi.org/10.1074/jbc.M110.151845">10.1074/jbc.M110.151845</a>.</div>
</div>
<div id="ref-mueller12" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Mueller, G.M., Yan, W., Copelovitch, L., Jarman, S., Wang, Z., Kinlough, C.L., Tolino, M.A., Hughey, R.P., Kleyman, T.R., and Rubenstein, R.C. (2012). Multiple residues in the distal <span>C</span> terminus of the α-subunit have roles in modulating human epithelial sodium channel activity. American Journal of Physiology-Renal Physiology <em>303</em>, F220–F228. <a href="https://doi.org/10.1152/ajprenal.00493.2011">10.1152/ajprenal.00493.2011</a>.</div>
</div>
<div id="ref-snyder2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Snyder, P.M., Bucher, D.B., and Olson, D.R. (2000). Gating <span>Induces</span> a <span>Conformational Change</span> in the <span>Outer Vestibule</span> of <span>Enac</span>. Journal of General Physiology <em>116</em>, 781–790. <a href="https://doi.org/10.1085/jgp.116.6.781">10.1085/jgp.116.6.781</a>.</div>
</div>
<div id="ref-kashlan2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Kashlan, O.B., Boyd, C.R., Argyropoulos, C., Okumura, S., Hughey, R.P., Grabe, M., and Kleyman, T.R. (2010). Allosteric inhibition of the epithelial <span>Na</span>+ channel through peptide binding at peripheral finger and thumb domains. J Biol Chem <em>285</em>, 35216–35223. <a href="https://doi.org/10.1074/jbc.M110.167064">10.1074/jbc.M110.167064</a>.</div>
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/references.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">References</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/appendices/processing-flow.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Image Processing</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"selector":".lightbox","openEffect":"zoom","closeEffect":"zoom","loop":true,"descPosition":"bottom"});</script>



</body></html>