<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>ENaC isn’t ASIC - 4&nbsp; ECD Stability Under Activating Conditions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/conclusion.html" rel="next">
<link href="../../parts/results/results_transmembrane-domain.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating nav-fixed slimcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../figures/other/domain-cartoon_ECD.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">ENaC isn’t ASIC</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../defense/" rel="" target=""><i class="bi bi-projector" role="img">
</i> 
 <span class="menu-text">Defense Slides</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.pdf">
              <i class="bi bi-bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.docx">
              <i class="bi bi-bi-file-word pe-1"></i>
            Download Docx
            </a>
          </li>
      </ul>
    </div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Results and Discussion</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_activating-conditions.html"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Frontmatter</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../overview.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">ENaC - An Overview</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_enac-in-the-body.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">ENaC Physiology and Regulation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_enac-as-a-channel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">ENaC Currents, Structure, and Gating</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Results and Discussion</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_transmembrane-domain.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">The Transmembrane Domain Remains Elusive</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_activating-conditions.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/conclusion.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Conclusion</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/conclusion/conclusion-doc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Conclusion</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/mat-meth.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Materials and Methods</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/processing-flow.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Image Processing</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/validation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Map Validation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/code.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Code and Computation</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#ecd-conformation-is-unchanged-by-trypsin-cleavage" id="toc-ecd-conformation-is-unchanged-by-trypsin-cleavage" class="nav-link active" data-scroll-target="#ecd-conformation-is-unchanged-by-trypsin-cleavage"><span class="header-section-number">4.1</span> ECD conformation is unchanged by trypsin cleavage</a>
  <ul class="collapse">
  <li><a href="#a-strained-tyrosine-in-the-thumb" id="toc-a-strained-tyrosine-in-the-thumb" class="nav-link" data-scroll-target="#a-strained-tyrosine-in-the-thumb"><span class="header-section-number">4.1.1</span> A strained tyrosine in the thumb</a></li>
  </ul></li>
  <li><a href="#sodium-self-inhibition-does-not-rearrange-the-ecd" id="toc-sodium-self-inhibition-does-not-rearrange-the-ecd" class="nav-link" data-scroll-target="#sodium-self-inhibition-does-not-rearrange-the-ecd"><span class="header-section-number">4.2</span> Sodium self-inhibition does not rearrange the ECD</a></li>
  <li><a href="#glycans-make-important-contacts-among-subunits-and-with-the-membrane" id="toc-glycans-make-important-contacts-among-subunits-and-with-the-membrane" class="nav-link" data-scroll-target="#glycans-make-important-contacts-among-subunits-and-with-the-membrane"><span class="header-section-number">4.3</span> Glycans make important contacts among subunits and with the membrane</a></li>
  <li><a href="#how-does-this-change-our-understanding-of-the-ecd" id="toc-how-does-this-change-our-understanding-of-the-ecd" class="nav-link" data-scroll-target="#how-does-this-change-our-understanding-of-the-ecd"><span class="header-section-number">4.4</span> How does this change our understanding of the ECD?</a>
  <ul class="collapse">
  <li><a href="#a-misfolded-tmd-may-uncouple-ecd-motions" id="toc-a-misfolded-tmd-may-uncouple-ecd-motions" class="nav-link" data-scroll-target="#a-misfolded-tmd-may-uncouple-ecd-motions"><span class="header-section-number">4.4.1</span> A misfolded TMD may uncouple ECD motions</a></li>
  <li><a href="#a-role-for-the-β9-α4-linker-in-gating" id="toc-a-role-for-the-β9-α4-linker-in-gating" class="nav-link" data-scroll-target="#a-role-for-the-β9-α4-linker-in-gating"><span class="header-section-number">4.4.2</span> A role for the β9-α4 linker in gating</a></li>
  <li><a href="#the-acidic-pockets-uncertain-role-in-sodium-self-inhibition" id="toc-the-acidic-pockets-uncertain-role-in-sodium-self-inhibition" class="nav-link" data-scroll-target="#the-acidic-pockets-uncertain-role-in-sodium-self-inhibition"><span class="header-section-number">4.4.3</span> The acidic pocket’s uncertain role in sodium self-inhibition</a></li>
  <li><a href="#glycans-may-further-differentiate-enac-subunits" id="toc-glycans-may-further-differentiate-enac-subunits" class="nav-link" data-scroll-target="#glycans-may-further-differentiate-enac-subunits"><span class="header-section-number">4.4.4</span> Glycans may further differentiate ENaC subunits</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span id="sec-results_ecd" class="quarto-section-identifier"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>Despite the fact that the TMD remains elusive, my hope was that investigating conditions thought to open the channel I might observe large-scale ECD rearrangements like those observed in ASIC. In this chapter I cover my efforts on that front, starting with a map of mouse ENaC solved pre- and post-trypsin treatment. Next, I investigate a putative sodium self-inhibition site that had been described previously in the literature (including by our own lab) in the presence of Na<sup>+</sup> and K<sup>+</sup>. Finally, I highlight the extensive newly-resolvable surface glycosylation present in all of my ECD maps.</p>
<section id="ecd-conformation-is-unchanged-by-trypsin-cleavage" class="level2 page-columns page-full" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="ecd-conformation-is-unchanged-by-trypsin-cleavage"><span class="header-section-number">4.1</span> ECD conformation is unchanged by trypsin cleavage</h2>
<div id="fig-trypsin-cartoon" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/trypsin-western/cleavage-cartoon.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="In this section I show that cleavage of mouse ENaC with trypsin does not induce any detectable rearrangement of the ECD"><img src="figures/trypsin-western/cleavage-cartoon.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.1: In this section I show that cleavage of mouse ENaC with trypsin does not induce any detectable rearrangement of the ECD</figcaption>
</figure>
</div>
<div class="page-columns page-full"><p>ENaC can reliably be opened by trypsin treatment<span class="citation" data-cites="caldwell2004"><sup><a href="../references.html#ref-caldwell2004" role="doc-biblioref">1</a></sup></span>. Additionally, the inhibitory peptide is easily resolved in all of my ECD maps. I wondered whether treatment with trypsin would result in movement of the finger and thumb domains as the model of ASIC-like gating movements predicts (<a href="#fig-trypsin-cartoon">Figure&nbsp;<span>4.1</span></a>). It’s important to note here that I had significant help from Arpita Bharadwaj, who performed the purification and initial characterization of the mouse material for these grids. Both uncleaved and trypsin-treated mouse ENaC were of similar map quality (3.24 and 3.30 Å respectively with whole-protein masks).</p><div class="no-row-height column-margin column-container"><span class="">Recall that when protons bind ASIC’s acidic pocket, the finger and thumb move approximately 8&nbsp;Å closer together<span class="citation" data-cites="yoder2018"><sup><a href="../references.html#ref-yoder2018" role="doc-biblioref">2</a></sup></span>.</span></div></div>
<div id="fig-cleavage-biochemistry" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/trypsin-western/western-and-silver.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Purified ENaC is cleaved by trypsin. A: Western blot for α (left) and γ (right) ENaC. Purified mouse protein was treated with trypsin and aprotinin as indicated. Bands corresponding to full-length subunits are marked with an asterisk. B: Silver stain of material with and without trypsin treatment."><img src="figures/trypsin-western/western-and-silver.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.2: Purified ENaC is cleaved by trypsin. <strong>A</strong>: Western blot for α (left) and γ (right) ENaC. Purified mouse protein was treated with trypsin and aprotinin as indicated. Bands corresponding to full-length subunits are marked with an asterisk. <strong>B</strong>: Silver stain of material with and without trypsin treatment.</figcaption>
</figure>
</div>
<div id="fig-cleavage-oocytes" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/trypsin-western/oocytes.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Trypsin treatment increases amiloride-sensitive currents in oocytes. Left: representative amiloride-sensitive current before (cyan) and after (orange) trypsin treatment. Only Na+ current is shown. Right: Average currents for all eight oocytes measured. All currents are scaled to pre-trypsin Na+ current to account for differential cell size and expression. Error bars represent 95% confidence interval. Values for individual oocytes are plotted as lines."><img src="figures/trypsin-western/oocytes.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.3: Trypsin treatment increases amiloride-sensitive currents in oocytes. <strong>Left</strong>: representative amiloride-sensitive current before (cyan) and after (orange) trypsin treatment. Only Na<sup>+</sup> current is shown. <strong>Right</strong>: Average currents for all eight oocytes measured. All currents are scaled to pre-trypsin Na<sup>+</sup> current to account for differential cell size and expression. Error bars represent 95% confidence interval. Values for individual oocytes are plotted as lines.</figcaption>
</figure>
</div>
<div id="fig-cleavage-maps" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/trypsin-western/maps-only.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="cryoEM maps of mouse ENaC γ (left) and α (right) subunit GRIP domains without (yellow) and with (purple) trypsin treatment. The inhibitory peptide is colored in each map."><img src="figures/trypsin-western/maps-only.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.4: cryoEM maps of mouse ENaC γ (left) and α (right) subunit GRIP domains without (yellow) and with (purple) trypsin treatment. The inhibitory peptide is colored in each map.</figcaption>
</figure>
</div>
<p>Biochemical analyses show that trypsin treatment cleaves both the α and γ subunits (<a href="#fig-cleavage-biochemistry">Figure&nbsp;<span>4.2</span></a>). Additionally, trypsin treatment of <em>X. laevis</em> oocytes expressing mouse αβγENaC show the expected increase in amiloride sensitive current (<a href="#fig-cleavage-oocytes">Figure&nbsp;<span>4.3</span></a>). However, aside from the disappearance of the inhibitory peptide, the ECD is otherwise superimposable (RMSD 0.440&nbsp;Å; <a href="#fig-cleavage-maps">Figure&nbsp;<span>4.4</span></a>). The finger and thumb do not shift even a fraction of what the ASIC-like model would predict.</p>
<p>Based on the observed cryoEM map density, only the γ subunit GRIP domain is released from the channel. However, biochemical characterization indicates both are digested by trypsin. I also note that the entire GRIP domain, not just the inhibitory peptide, has reduced map density in the trypsin-treated sample. This suggests that the entire grip domain might be released from the subunit, but of course it’s equally possible that release of the inhibitory peptide results in increased GRIP flexibility, which blurs out the map.</p>
<p>These results are incompatible with a large-scale movement of the ECD upon trypsin digestion. The existence of some γ GRIP density indicates that the trypsin-treated map is an ensemble of particles which have and have not released the γ GRIP domain. However, I do not observe any blurring of the γ thumb, which would be expected in a map comprising particles with significantly shifted conformations. I must therefore conclude that the removal of the γ inhibitory peptide is not sufficient, on its own, to induce such changes. Additionally, I note that these maps are the first indication that trypsin processing of the α subunit does not release its GRIP domain. This is surprising but perhaps not wholly unexpected; the γ subunit is known to be essential for activation by trypsin but the same has not yet been shown for αENaC<span class="citation" data-cites="diakov2008"><sup><a href="../references.html#ref-diakov2008" role="doc-biblioref">3</a></sup></span>.</p>
<section id="a-strained-tyrosine-in-the-thumb" class="level3" data-number="4.1.1">
<h3 data-number="4.1.1" class="anchored" data-anchor-id="a-strained-tyrosine-in-the-thumb"><span class="header-section-number">4.1.1</span> A strained tyrosine in the thumb</h3>
<div id="fig-tyr-rotamer" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/tyr-rotamer/all-tyr-figure.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="A strained tyrosine rotamer is present in the γ subunit is released by trypsin cleavage. A: Human CKO γY425 (right), with approximate position indicated by a star in the cartoon (left). Hydrogen bonds are colored blue for ideal geometry and orange for slightly strained distance. B: γY425 (Y431 in mouse) colored by construct: human CKO, human CKO/DEG, mouse uncleaved, and mouse trypsin as red, blue, yellow, and purple respectively. C: αY447 (Y474 in mouse) colored by construct. D: Comparison of the inhibitory peptides of α (blue), β (red), and γ (magenta). The displayed model is human CKO digitonin, but the peptides are superimposable in all six models."><img src="figures/tyr-rotamer/all-tyr-figure.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.5: A strained tyrosine rotamer is present in the γ subunit is released by trypsin cleavage. <strong>A</strong>: Human CKO γY425 (right), with approximate position indicated by a star in the cartoon (left). Hydrogen bonds are colored blue for ideal geometry and orange for slightly strained distance. <strong>B</strong>: γY425 (Y431 in mouse) colored by construct: human CKO, human CKO/DEG, mouse uncleaved, and mouse trypsin as red, blue, yellow, and purple respectively. <strong>C</strong>: αY447 (Y474 in mouse) colored by construct. <strong>D</strong>: Comparison of the inhibitory peptides of α (blue), β (red), and γ (magenta). The displayed model is human CKO digitonin, but the peptides are superimposable in all six models.</figcaption>
</figure>
</div>
<div id="fig-tyr-rotamer-maps" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/tyr-rotamer/supplement-maps.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="CryoEM map potential for strained γENaC tyrosine for all maps/models. The maps are masked around the residue of interest."><img src="figures/tyr-rotamer/supplement-maps.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.6: CryoEM map potential for strained γENaC tyrosine for all maps/models. The maps are masked around the residue of interest.</figcaption>
</figure>
</div>
<p>The high-resolution maps of the γ ECD reveal an interesting rotamer outlier, γY425 (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>4.5</span></a>). The map for this residue is of high quality in five of the six maps, supporting its unlikely (prior probability of 0.035%) conformation (<a href="#fig-tyr-rotamer-maps">Figure&nbsp;<span>4.6</span></a>). The single map with poor quality is the trypsin-treated mouse map, in which the density is blurred and unclear. Human γY425 forms hydrogen bonds with the finger (H233) and inhibitory peptide (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>4.5</span></a> A). This tyrosine adopts the same pose in all models <em>except</em> the trypsin-treated mouse model, in which it may rotate to point toward the outer GRIP domain (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>4.5</span></a> B). Given the poor quality of γY431 (the equivalent position to human Y425) in the mouse trypsin map, I cannot assign this rotamer with certainty.</p>
<p>The equivalent residue in α is also a tyrosine (αY447 and 474 in human and mouse respectively), but the inhibitory peptide curves away from the channel center, removing the interactions with the backbone and aromatic oxygen atoms and allowing a more favored rotamer (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>4.5</span></a> C and D). The same position in β, N417, only forms a hydrogen bond with the finger domain (N225). The lack of any interactions between βN417 and the βENaC GRIP domain, which is not cleaved to open the channel, is further suggestive of this residue’s role in proteolytic gating.</p>
<p>I note also that the map quality worsens for the nearby γM438, mutation of which alters channel gating<span class="citation" data-cites="maarouf2009"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a></sup></span>. The reduced quality for this map region upon trypsin treatment may represent increased flexibility, perhaps indicating that conformational strain in the upper thumb is released along with the inhibitory peptide.</p>
</section>
</section>
<section id="sodium-self-inhibition-does-not-rearrange-the-ecd" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="sodium-self-inhibition-does-not-rearrange-the-ecd"><span class="header-section-number">4.2</span> Sodium self-inhibition does not rearrange the ECD</h2>
<p>I next investigated differences in ECD conformation among protein purified in external K<sup>+</sup> instead of Na<sup>+</sup>. High extracellular K<sup>+</sup> does not prime inhibition in wild-type channels and SSI is rapidly attenuated upon exposure to low (i.e., &lt;10&nbsp;mM) Na<sup>+</sup><span class="citation" data-cites="maarouf2009 kashlan2015"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a>,<a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a></sup></span>. Protein purified in K<sup>+</sup> buffer with nominally zero Na<sup>+</sup> should therefore be completely uninhibited.</p>
<p>The linker connecting β6 and β7 is important for SSI, and our lab has previously reported evidence of a putative bound Na<sup>+</sup> ion in this region<span class="citation" data-cites="kashlan2015 noreng20"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a>,<a href="../references.html#ref-noreng20" role="doc-biblioref">6</a></sup></span>. Moreover, cross-linking of several pairs of amino acids in the ECD affects SSI magnitude, implying that some rearrangement is required for SSI<span class="citation" data-cites="kashlan2015 zhang2023"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a>,<a href="../references.html#ref-zhang2023" role="doc-biblioref">7</a></sup></span>. Given this body of research, I solved structures of ENaC in high (200&nbsp;mM) and low (nominal 0&nbsp;mM&nbsp;Na<sup>+</sup>, 200&nbsp;mM&nbsp;K<sup>+</sup>) sodium. I expected the high-Na<sup>+</sup> model to show rearrangements throughout the ECD, as would be required to transfer Na<sup>+</sup>-binding state to the pore.</p>
<p>Our maps of human CKO ENaC in Na<sup>+</sup> and K<sup>+</sup> buffers achieved resolutions of 2.3 and 3.0&nbsp;Å, respectively. The Na<sup>+</sup> structure was solved in digitonin with both the α- and β-binding Fabs, while the K<sup>+</sup> structure was solved in nanodiscs without any Fabs. Although previously it has proven difficult to break the pseudo-threefold symmetry without including Fabs, I worried that their inclusion may affect SSI. Fortunately, the high quality of the cryoEM data allowed us to break the symmetry with neither Fabs nor an external reference — I believe the major symmetry breaking features are micelle-proximal glycans (<a href="#fig-fab-compare">Figure&nbsp;<span>4.7</span></a>). Binding of either Fab induces minimal rearrangement of ENaC; the β epitope does not change conformation upon binding, while the α epitope pulls the loop from αP264-T274 approximately 4.5&nbsp;Å away from the protein core without distorting the structure outside that loop.</p>
<div id="fig-fab-compare" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/fabs/fabs.png" class="lightbox" data-gallery="quarto-lightbox-gallery-7" title="Fab binding induces only minor changes in the extracellular domain of ENaC. A: Human CKO/DEG map. A low-pass filtered map is outlined to show pseudosymmetry-breaking glycosylation sites (arrows). B: the α subunit, C: the β subunit. Human CKO nanodisc model is colored by subunit, the human CKO/DEG ENaC is white, and human CKO/DEG Fabs are grey."><img src="figures/fabs/fabs.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.7: Fab binding induces only minor changes in the extracellular domain of ENaC. A: Human CKO/DEG map. A low-pass filtered map is outlined to show pseudosymmetry-breaking glycosylation sites (arrows). B: the α subunit, C: the β subunit. Human CKO nanodisc model is colored by subunit, the human CKO/DEG ENaC is white, and human CKO/DEG Fabs are grey.</figcaption>
</figure>
</div>
<div id="fig-sodium-self-inhib" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/sodium-self-inhibition/ssi.png" class="lightbox" data-gallery="quarto-lightbox-gallery-8" title="Presence or absence of Na+ does not rearrange the ECD. A: A previously identified ion-binding site still shows cryoEM map density with both Na+ (left) and K+ (right). The green dot corresponds to the previously-reported (PDB 6WTH) putative ion position. B: The β6-β7 linker in Na+ or K+ buffers."><img src="figures/sodium-self-inhibition/ssi.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.8: Presence or absence of Na<sup>+</sup> does not rearrange the ECD. <strong>A</strong>: A previously identified ion-binding site still shows cryoEM map density with both Na<sup>+</sup> (left) and K<sup>+</sup> (right). The green dot corresponds to the previously-reported (PDB 6WTH) putative ion position. <strong>B</strong>: The β6-β7 linker in Na<sup>+</sup> or K<sup>+</sup> buffers.</figcaption>
</figure>
</div>
<p>To my surprise, no resolved region of the ECD undergoes any significant structural rearrangement between high- and low-Na<sup>+</sup> buffers (RMSD 0.490&nbsp;Å, <a href="#fig-sodium-self-inhib">Figure&nbsp;<span>4.8</span></a> A). Aside from the changes induced by the α-binding Fab, the only Cα shifts greater than 2 Å are residues terminating the resolved regions of the channel, which are likely a result of poor map quality. An acidic cleft in the αENaC finger, near the β-ball, is implicated in sodium self-inhibition<span class="citation" data-cites="kashlan2015"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a></sup></span>. However, the cryoEM maps have density for a ligand in the acidic cleft in both the Na<sup>+</sup> and K<sup>+</sup> maps (<a href="#fig-sodium-self-inhib">Figure&nbsp;<span>4.8</span></a> A), although the site is slightly expanded in the K<sup>+</sup> map, perhaps to accommodate the larger K<sup>+</sup> ion. The β6-β7 linker has high quality density in both maps, supporting an almost-identical (Cα RMSD 1.076&nbsp;Å when the whole α subunit is aligned by the palm domain) conformation, aside from the blurring of αR333 across multiple rotamers (<a href="#fig-sodium-self-inhib">Figure&nbsp;<span>4.8</span></a> B).</p>
<p>Although SSI is sensitive to pH, the buffer conditions for the CKO map should be at least 55% inhibited<span class="citation" data-cites="collier2009"><sup><a href="../references.html#ref-collier2009" role="doc-biblioref">8</a></sup></span>. Attempts at detecting other classes of particle in the CKO ENaC (Na<sup>+</sup>) map were not successful, leading us to believe that the particles are homogeneous. The lack of any significant structural difference in the acidic pocket in the presence of Na<sup>+</sup> and K<sup>+</sup> is more compatible with an SSI model in which Na<sup>+</sup>-bound ENaC is “locked” in a closed state, rather than some third inhibited state.</p>
</section>
<section id="glycans-make-important-contacts-among-subunits-and-with-the-membrane" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="glycans-make-important-contacts-among-subunits-and-with-the-membrane"><span class="header-section-number">4.3</span> Glycans make important contacts among subunits and with the membrane</h2>
<p>Glycosylation of ENaC at conserved asparagine sites is essential for sodium self-inhibition, trypsin activation, and mechanosensitivity<span class="citation" data-cites="kashlan2018 knoepp2020"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">9</a>,<a href="../references.html#ref-knoepp2020" role="doc-biblioref">10</a></sup></span>. The high-quality maps obtained in this study enable modeling of significantly more and longer glycans than in previous work. I find a total of 19 distinct glycosylation sites, with up to nine modellable sugars in a single glycan.</p>
<p>For the most part, the sugars are identical in all six models and make no protein or glycan interactions. These sugars likely play a role in other regulatory roles, such as channel trafficking and maturation or interaction with the extracellular matrix<span class="citation" data-cites="kashlan2018 hughey04 knoepp2020"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">9</a>–<a href="../references.html#ref-hughey04" role="doc-biblioref">11</a></sup></span>. However, two classes of glycan are particularly interesting: the micelle-proximal and the intersubunit glycans.</p>
<div id="fig-intersub-glyc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/intersubunit-glyc/intersub-glyc.png" class="lightbox" data-gallery="quarto-lightbox-gallery-9" title="ENaC’s glycosylation sites form important intersubunit interactions. A: Cartoon overview of intersubunit glycosylations. Dotted lines represent glycan chains. Hexagons represent sugars, circles represent potentially-interacting residues. Unfiltered maps for the first two sugars are presented to indicate confidence in Asn assignment. B: Silver-stained SDS-PAGE gel of protein treated with PNGase F. C: Intersubunit glycan from βN260 to αD235, K495. D: Intersubunit interaction between glycans from βN207 (red) and γN271 (magenta). In both C and D, the solid surface is a gaussian-blurred version of the sharp human CKO digitonin map (mesh)."><img src="figures/intersubunit-glyc/intersub-glyc.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.9: ENaC’s glycosylation sites form important intersubunit interactions. <strong>A</strong>: Cartoon overview of intersubunit glycosylations. Dotted lines represent glycan chains. Hexagons represent sugars, circles represent potentially-interacting residues. Unfiltered maps for the first two sugars are presented to indicate confidence in Asn assignment. <strong>B</strong>: Silver-stained SDS-PAGE gel of protein treated with PNGase F. <strong>C</strong>: Intersubunit glycan from βN260 to αD235, K495. <strong>D</strong>: Intersubunit interaction between glycans from βN207 (red) and γN271 (magenta). In both C and D, the solid surface is a gaussian-blurred version of the sharp human CKO digitonin map (mesh).</figcaption>
</figure>
</div>
<p>The intersubunit glycans stretch from one subunit to another (<a href="#fig-intersub-glyc">Figure&nbsp;<span>4.9</span></a> A). I am able to model long chains for these glycans, with some maps supporting up to nine monosaccharides. Due to the flexibility of these long chains, the terminal side is quite blurred out in the cryoEM maps, and so I cannot be confident about the exact orientation or identity of the constituent sugars. Nonetheless, the maps are of sufficient quality to at least suggest sugar-mediated interactions between subunits, and the first sugars are of sufficient resolution to assign the asparagine residues to which they are coupled.</p>
<p>Treatment with PNGase F induces a size shift in the purified protein, confirming that the protein is glycosylated (<a href="#fig-intersub-glyc">Figure&nbsp;<span>4.9</span></a> B). I built typical high-mannose glycans (Man<sub>5</sub>GlcNAc<sub>2</sub>) into the models as expected for GnTI<sup>-</sup> and Sf9 cells to assess these interactions at a low resolution<span class="citation" data-cites="reeves2002 shi2007"><sup><a href="../references.html#ref-reeves2002" role="doc-biblioref">12</a>,<a href="../references.html#ref-shi2007" role="doc-biblioref">13</a></sup></span>. One glycan reaches from βN260 to the α subunit (<a href="#fig-intersub-glyc">Figure&nbsp;<span>4.9</span></a> C), potentially hydrogen bonding with αD235 and αK459. These three residues are absolutely conserved across ENaC genes surveyed. Similarly, glycans attached to conserved sites βN207 and γN271 connect the β GRIP to the γ palm (<a href="#fig-intersub-glyc">Figure&nbsp;<span>4.9</span></a> D). Knock-out of sugars in similar positions in αENaC increase blood pressure in mice<span class="citation" data-cites="knoepp2020"><sup><a href="../references.html#ref-knoepp2020" role="doc-biblioref">10</a></sup></span>. The relatively high quality of these glycans suggest they are quite conformationally stable and may play a role in channel function.</p>
<div id="fig-micelle-glyc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/membrane-glyc/membrane-glyc.png" class="lightbox" data-gallery="quarto-lightbox-gallery-10" title="ENaC’s glycans contact the micelle. A: An overview of the position of the micelle-proximal glycans and the micelle. The image on the right is a zoomed view of the boxed region on the left. B and C: The sharpened map of human CKO digitonin for the first sugar attached to αN397 and βN364, respectively. D: In-grid rotation distribution for two selected classes of CKO nanodiscs. Note that both classes have even distribution of in-plane angles, indicating that the micelle tilt does not align with some grid force such as fluid flow during blotting. E: A cartoon representation of the position of the membrane-proximal glycans relative to the micelle tilt axis."><img src="figures/membrane-glyc/membrane-glyc.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.10: ENaC’s glycans contact the micelle. <strong>A</strong>: An overview of the position of the micelle-proximal glycans and the micelle. The image on the right is a zoomed view of the boxed region on the left. <strong>B and C</strong>: The sharpened map of human CKO digitonin for the first sugar attached to αN397 and βN364, respectively. <strong>D</strong>: In-grid rotation distribution for two selected classes of CKO nanodiscs. Note that both classes have even distribution of in-plane angles, indicating that the micelle tilt does not align with some grid force such as fluid flow during blotting. <strong>E</strong>: A cartoon representation of the position of the membrane-proximal glycans relative to the micelle tilt axis.</figcaption>
</figure>
</div>
<p>The micelle-proximal glycans are anchored to hαN397 and hβN364 and the equivalent mouse residues, mαN424 and mβN362 (<a href="#fig-micelle-glyc">Figure&nbsp;<span>4.10</span></a> A). The α site is conserved in mammalian ENaCs, while the β site is conserved in all surveyed organisms except lungfish. These glycans extend away from the ECD body and turn down toward the micelle or lipids and are clear in all ENaC maps solved to date. The γ subunit does not have an equivalent asparagine residue, and so does not have a micelle-proximal glycan.</p>
<p>I note that all ENaC maps produced to date also show a consistent tilt of the micelle or nanodisc such that the γ subunit is closer to the surface of the micelle than are the α or β subunits; this tilt does not appear to be an artifact of in-plane rotation (<a href="#fig-micelle-glyc">Figure&nbsp;<span>4.10</span></a> B). I propose therefore that this tilt may in part be caused by the presence of the micelle-proximal glycans, which are only missing from γENaC (<a href="#fig-micelle-glyc">Figure&nbsp;<span>4.10</span></a> C). Their presence in the nanodisc map may indicate that they perform a similar role in a native cell membrane context.</p>
</section>
<section id="how-does-this-change-our-understanding-of-the-ecd" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="how-does-this-change-our-understanding-of-the-ecd"><span class="header-section-number">4.4</span> How does this change our understanding of the ECD?</h2>
<section id="a-misfolded-tmd-may-uncouple-ecd-motions" class="level3" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="a-misfolded-tmd-may-uncouple-ecd-motions"><span class="header-section-number">4.4.1</span> A misfolded TMD may uncouple ECD motions</h3>
<p>Regardless of the treatment applied, I cannot measure any change in the ECD. This stability stands in stark contrast to the highly dynamic ECD of the gating model widely assumed in the field and based on that of ASIC1<span class="citation" data-cites="kashlan11 yoder2018"><sup><a href="../references.html#ref-yoder2018" role="doc-biblioref">2</a>,<a href="../references.html#ref-kashlan11" role="doc-biblioref">14</a></sup></span>. I observe trypsin cleavage of both the α and γ GRIP domains, but only reduced cryoEM map density for the γ domain. The maps are otherwise perfectly superimposable, with unchanged ECDs and TMDs. Loss of density for only the γ GRIP domain is surprising but not entirely unexpected — channels lacking the γ subunit or specific lysine residues in the γ ECD are not activated by trypsin<span class="citation" data-cites="diakov2008"><sup><a href="../references.html#ref-diakov2008" role="doc-biblioref">3</a></sup></span>. It is therefore possible that while the α GRIP domain is cleaved, it is not directly involved in trypsin-mediated channel activation. Based on biochemical data, I speculate that the α GRIP domain is also cleaved by trypsin, but the GRIP domain is still clearly visible in our cryoEM maps. In any case, a lack of any observable channel rearrangement upon removal of the γ inhibitory peptide (a state in which ENaC’s P<sub>O</sub> approaches 0.9) must lead us to a deeper consideration of ENaC gating<span class="citation" data-cites="bruns07"><sup><a href="../references.html#ref-bruns07" role="doc-biblioref">15</a></sup></span>.</p>
<p>Although the best-studied mechanism of ENaC opening is GRIP proteolysis, multiple studies have found other stimuli which open the channel, including laminar shear stress and phospholipids (see <a href="../introduction/intro_enac-as-a-channel.html#sec-intro_direct-reg"><span>Section&nbsp;2.3</span></a> for more detail). ENaC P<sub>O</sub> is also sensitive to changes in both intracellular and extracellular pH<span class="citation" data-cites="chalfant1999"><sup><a href="../references.html#ref-chalfant1999" role="doc-biblioref">16</a></sup></span>. It is thus possible that the cell possesses multiple mechanisms with which it can open ENaC and uses them in combination to tune Na<sup>+</sup> permeability.</p>
<p>The possibility of a more complex pathway to open ENaC is supported by <em>in vivo</em> studies as well. The proportion of cleaved γENaC from mice lacking SGK1 is significantly smaller than that of WT mice, but cells from these kidneys have indistinguishable amiloride-sensitive current<span class="citation" data-cites="fejestoth08"><sup><a href="../references.html#ref-fejestoth08" role="doc-biblioref">17</a></sup></span>. Similarly, ENaC missing the canonical furin cleavage site in the γ subunit is not cleaved when expressed in <em>X. laevis</em> oocytes, but currents do not differ between the mutant channels and WT<span class="citation" data-cites="harris2008"><sup><a href="../references.html#ref-harris2008" role="doc-biblioref">18</a></sup></span>.</p>
<p>Indeed, in different tissues and in different studies the cleavage of either α- or γENaC seems to be of greater importance, hinting at an environmental effect on the relationship between GRIP cleavage and channel P<sub>O</sub><span class="citation" data-cites="hughey2004 bruns07 caldwell2005"><sup><a href="../references.html#ref-bruns07" role="doc-biblioref">15</a>,<a href="../references.html#ref-hughey2004" role="doc-biblioref">19</a>,<a href="../references.html#ref-caldwell2005" role="doc-biblioref">20</a></sup></span>. Based on this prior research and the maps I present here, I suspect that ENaC gating is not as simple as the proposed mechanism in which a single, coordinated protein maturation step moves the channel from a low- to high-P<sub>O</sub> state<span class="citation" data-cites="hughey03"><sup><a href="../references.html#ref-hughey03" role="doc-biblioref">21</a></sup></span>. However, I cannot make firm conclusions about ENaC gating due to the unresolved question of the TMD.</p>
<p>As discussed in <a href="results_transmembrane-domain.html"><span>Chapter&nbsp;3</span></a>, the TMDs of all six models are disturbed and likely do not represent a physiological conformation. It is possible that the ECD requires some kind of feedback from the TMD, and that this coupling has been lost during purification and grid preparation.</p>
<p>Mutations in linkers between the TMD and ECD of other channels have dramatic effects on ECD function; for instance, the <em>lurcher</em> mutation in the GluR1 ECD/TMD linker dramatically increases ECD affinity for glutamate<span class="citation" data-cites="taverna2000"><sup><a href="../references.html#ref-taverna2000" role="doc-biblioref">22</a></sup></span>. I suspect that the ENaC ECD requires a fully-assembled six-TM pore region to respond to GRIP domain excision, but such a conclusion would require deeper investigation. A TMD/ECD decoupling would also explain the lack of ECD shift with the introduction of the DEG mutation, which has been shown to hold the channel in an open state, and for which I see dramatic TMD restructuring but no change in the ECD<span class="citation" data-cites="snyder2000"><sup><a href="../references.html#ref-snyder2000" role="doc-biblioref">23</a></sup></span>.</p>
<p>ENaC does not seem amenable to the standard means of membrane protein purification. It is possible that finding a more-stable ortholog would improve the quality of the TMD, as our mouse maps have significantly better TMD density than our human maps.</p>
</section>
<section id="a-role-for-the-β9-α4-linker-in-gating" class="level3" data-number="4.4.2">
<h3 data-number="4.4.2" class="anchored" data-anchor-id="a-role-for-the-β9-α4-linker-in-gating"><span class="header-section-number">4.4.2</span> A role for the β9-α4 linker in gating</h3>
<p>In <em>Malacoceros fuliginosus</em> FaNaC, a histidine in the β9-α4 linker forms contacts with both TMs<span class="citation" data-cites="valeriakalienkova2023"><sup><a href="../references.html#ref-valeriakalienkova2023" role="doc-biblioref">24</a></sup></span>. This interaction blocks channel opening until ligand is bound. In ENaC and <em>Helix aspersa</em> FaNaC the equivalent residue is a tyrosine, while in ASIC it is a tryptophan<span class="citation" data-cites="lingueglia1995"><sup><a href="../references.html#ref-lingueglia1995" role="doc-biblioref">25</a></sup></span>. It is possible that any bulky or aromatic residue plays a similar ECD-lock role in this β-turn. In my models, only the β-subunit tyrosine is close enough to hydrogen bond with the TMs as I have modeled them, but this could be an artifact of the disordered TMD, or the interaction may be mediated by lipids.</p>
<p>I also note that the proline directly preceding this tyrosine in both the β and γ loops (α does not have a proline in this loop) is in the <em>cis</em> conformation. This is true in both human and mouse ENaC, in all treatment conditions. Our previous models missed this conformer, likely because of significantly poorer resolution. ASIC’s tryptophan in this position is also preceded by a <em>cis</em>-proline<span class="citation" data-cites="yoder2018"><sup><a href="../references.html#ref-yoder2018" role="doc-biblioref">2</a></sup></span>. In light of the important role this loop plays in FaNaC gating, and the conserved <em>cis</em>-Pro/aromatic pair in ASIC and ENaC, further investigation of this loop may prove fruitful.</p>
</section>
<section id="the-acidic-pockets-uncertain-role-in-sodium-self-inhibition" class="level3" data-number="4.4.3">
<h3 data-number="4.4.3" class="anchored" data-anchor-id="the-acidic-pockets-uncertain-role-in-sodium-self-inhibition"><span class="header-section-number">4.4.3</span> The acidic pocket’s uncertain role in sodium self-inhibition</h3>
<p>Previous reports suggested that rearrangement of an acidic pocket in the α finger plays a key role in SSI<span class="citation" data-cites="kashlan2015"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a></sup></span>. It is therefore highly surprising that this pocket does not change upon release of Na<sup>+</sup>. However, there are species differences in ENaC SSI, and the functional work describing this acidic pocket was all performed on mouse ENaC, while our high- and low-Na<sup>+</sup> models are both of human ENaC<span class="citation" data-cites="collier2009 kashlan2015 maarouf2009"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a>,<a href="../references.html#ref-kashlan2015" role="doc-biblioref">5</a>,<a href="../references.html#ref-collier2009" role="doc-biblioref">8</a></sup></span>.</p>
<p>In human (but not rat) ENaC, pH titration changes the magnitude of SSI but, crucially, not the K<sub>i</sub> of Na<sup>+</sup>; a surprising result if surface-accessible acidic residues form the SSI Na<sup>+</sup>-binding site. Moreover, ENaC cleavage essentially abrogates SSI, and residues in the upper finger which interact with the GRIP domain are implicated in SSI<span class="citation" data-cites="sheng06 bize07 maarouf2009"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a>,<a href="../references.html#ref-sheng06" role="doc-biblioref">26</a>,<a href="../references.html#ref-bize07" role="doc-biblioref">27</a></sup></span>. Despite this relationship, I also see no conformational shift of the acidic pocket upon trypsin treatment.</p>
<p>It may therefore be that the acidic pocket simply has a higher affinity for Na<sup>+</sup> than K<sup>+</sup> and therefore presents a higher barrier to ECD conformational shift when bound to Na<sup>+</sup>. This would effectively trap the channel in a closed state, rather than some third sodium self-inhibited state. Without observing the complete gating cycle of ENaC and the concomitant conformational changes it is impossible to make any firm conclusions about the precise mechanism of SSI. I therefore hope that our models will serve as a useful guide in future studies on this acidic pocket, as well as a search for other potential sources of SSI.</p>
</section>
<section id="glycans-may-further-differentiate-enac-subunits" class="level3" data-number="4.4.4">
<h3 data-number="4.4.4" class="anchored" data-anchor-id="glycans-may-further-differentiate-enac-subunits"><span class="header-section-number">4.4.4</span> Glycans may further differentiate ENaC subunits</h3>
<p>These maps represent the first structural study of ENaC’s extensive glycosylation. I must emphasize that these models all arise from particles which have only high-mannose glycosylation. However, glycosylation is required for ENaC function and trypsin digestion and our channels have the expected electrophysiological properties and respond to trypsin treatment in the expression system (<a href="#fig-ephys">Figure&nbsp;<span>4.11</span></a>)<span class="citation" data-cites="kashlan2018"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">9</a></sup></span>. I am therefore confident that the high-mannose glycans are sufficient for mechanistic inference.</p>
<div id="fig-ephys" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/ephys/491-mean-iv.png" class="lightbox" data-gallery="quarto-lightbox-gallery-11" title="I/V curves for human CKO and CKO/DEG channels. Values are normalized per-cell to the inward current at -80 mV before trypsin treatment. Cells were treated with trypsin for five minutes before being re-recorded. Each point is the average of three cells. Error bars represent +/- one standard deviation."><img src="figures/ephys/491-mean-iv.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;4.11: I/V curves for human CKO and CKO/DEG channels. Values are normalized per-cell to the inward current at -80 mV before trypsin treatment. Cells were treated with trypsin for five minutes before being re-recorded. Each point is the average of three cells. Error bars represent +/- one standard deviation.</figcaption>
</figure>
</div>
<p>Two distinct classes of glycan make observable interactions in our models: membrane-proximal and intersubunit glycans. I propose that the membrane-proximal glycans may help establish the micelle/nanodisc tilt observed in all ENaC maps to date, and that this tilt may have as-yet unknown functional implications.</p>
<p>I am not suggesting that these glycans alone are sufficient to define ENaC’s orientation in the membrane For example, in addition to missing a membrane-proximal glycan γENaC has two fewer residues between the palm and TM1 than do the α or β subunits<span class="citation" data-cites="noreng18"><sup><a href="../references.html#ref-noreng18" role="doc-biblioref">28</a></sup></span>.</p>
<p>The β intersubunit glycan interacts with the top of the α thumb, which may serve to rigidify it. Indeed, knock-out of β glycans protects the α subunit from trypsin digestion<span class="citation" data-cites="kashlan2018"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">9</a></sup></span>. Perhaps the β intersubunit glycan holds the α subunit in a pose which primes it for proteolytic cleavage. In contrast, the γ palm, not thumb, is rigidified by the other large intersubunit glycan. I note also that previous observation of channels cleaved by endogenous cellular machinery indicated that removal of the inhibitory peptide reduces the map quality for γ- but not αP3 and P4<span class="citation" data-cites="noreng20"><sup><a href="../references.html#ref-noreng20" role="doc-biblioref">6</a></sup></span>.</p>
<p>This alternate rigidification is interesting in the light of sodium self-inhibition. As yet, there is not a clear explanation for how mutating αH255 and γH233 (occupying the same structural position in their respective subunits) might have opposite effects on SSI (augmenting and eliminating it, respectively)<span class="citation" data-cites="sheng2004"><sup><a href="../references.html#ref-sheng2004" role="doc-biblioref">29</a></sup></span>. Considering a rigid macro-domain comprising the uncleavable β subunit, γ palm, and α thumb (connected via glycans) and a flexible macro-domain comprising the α palm and finger and γ thumb, I see that γH233 belongs to the former, while αH255 belongs to the latter. Of course, more functional study is required to firmly establish this link, but I note that the SSI of channels lacking N-linked glycosylation sites is significantly reduced compared to WT<span class="citation" data-cites="kashlan2018"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">9</a></sup></span>.</p>


<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-caldwell2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Caldwell, R.A., Boucher, R.C., and Stutts, M.J. (2004). Serine protease activation of near-silent epithelial <span>Na</span>+ channels. American Journal of Physiology-Cell Physiology <em>286</em>, C190–C194. <a href="https://doi.org/10.1152/ajpcell.00342.2003">10.1152/ajpcell.00342.2003</a>.</div>
</div>
<div id="ref-yoder2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Yoder, N., Yoshioka, C., and Gouaux, E. (2018). Gating mechanisms of acid-sensing ion channels. Nature <em>555</em>, 397–401. <a href="https://doi.org/10.1038/nature25782">10.1038/nature25782</a>.</div>
</div>
<div id="ref-diakov2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Diakov, A., Bera, K., Mokrushina, M., Krueger, B., and Korbmacher, C. (2008). Cleavage in the γ-subunit of the epithelial sodium channel (<span>ENaC</span>) plays an important role in the proteolytic activation of near-silent channels. J Physiol <em>586</em>, 4587–4608. <a href="https://doi.org/10.1113/jphysiol.2008.154435">10.1113/jphysiol.2008.154435</a>.</div>
</div>
<div id="ref-maarouf2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Maarouf, A.B., Sheng, N., Chen, J., Winarski, K.L., Okumura, S., Carattino, M.D., Boyd, C.R., Kleyman, T.R., and Sheng, S. (2009). Novel <span>Determinants</span> of <span>Epithelial Sodium Channel Gating</span> within <span>Extracellular Thumb Domains</span> *. Journal of Biological Chemistry <em>284</em>, 7756–7765. <a href="https://doi.org/10.1074/jbc.M807060200">10.1074/jbc.M807060200</a>.</div>
</div>
<div id="ref-kashlan2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Kashlan, O.B., Blobner, B.M., Zuzek, Z., Tolino, M., and Kleyman, T.R. (2015). Na+ <span>Inhibits</span> the <span>Epithelial Na</span>+ <span>Channel</span> by <span>Binding</span> to a <span>Site</span> in an <span>Extracellular Acidic Cleft</span> *. Journal of Biological Chemistry <em>290</em>, 568–576. <a href="https://doi.org/10.1074/jbc.M114.606152">10.1074/jbc.M114.606152</a>.</div>
</div>
<div id="ref-noreng20" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Noreng, S., Posert, R., Bharadwaj, A., Houser, A., and Baconguis, I. (2020). Molecular principles of assembly, activation, and inhibition in epithelial sodium channel. eLife <em>9</em>, e59038. <a href="https://doi.org/10.7554/eLife.59038">10.7554/eLife.59038</a>.</div>
</div>
<div id="ref-zhang2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Zhang, L., Wang, X., Chen, J., Sheng, S., and Kleyman, T.R. (2023). Extracellular intersubunit interactions modulate epithelial <span>Na</span>+ channel gating. Journal of Biological Chemistry <em>299</em>, 102914. <a href="https://doi.org/10.1016/j.jbc.2023.102914">10.1016/j.jbc.2023.102914</a>.</div>
</div>
<div id="ref-collier2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Collier, D.M., and Snyder, P.M. (2009). Extracellular <span>Protons Regulate Human ENaC</span> by <span>Modulating Na</span>+ <span class="nocase">Self-inhibition</span> *. Journal of Biological Chemistry <em>284</em>, 792–798. <a href="https://doi.org/10.1074/jbc.M806954200">10.1074/jbc.M806954200</a>.</div>
</div>
<div id="ref-kashlan2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Kashlan, O.B., Kinlough, C.L., Myerburg, M.M., Shi, S., Chen, J., Blobner, B.M., Buck, T.M., Brodsky, J.L., Hughey, R.P., and Kleyman, T.R. (2018). N-linked glycans are required on epithelial <span>Na</span>(+) channel subunits for maturation and surface expression. Am. J. Physiol. Renal Physiol. <em>314</em>, F483–f492. <a href="https://doi.org/10.1152/ajprenal.00195.2017">10.1152/ajprenal.00195.2017</a>.</div>
</div>
<div id="ref-knoepp2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Knoepp, F., Ashley, Z., Barth, D., Baldin, J.-P., Jennings, M., Kazantseva, M., Saw, E.L., Katare, R., Alvarez de la Rosa, D., Weissmann, N., et al. (2020). Shear force sensing of epithelial <span>Na</span>+ channel (<span>ENaC</span>) relies on <span class="nocase">N-glycosylated</span> asparagines in the palm and knuckle domains of <span class="nocase">αENaC</span>. Proceedings of the National Academy of Sciences <em>117</em>, 717–726. <a href="https://doi.org/10.1073/pnas.1911243117">10.1073/pnas.1911243117</a>.</div>
</div>
<div id="ref-hughey04" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Hughey, R.P., Bruns, J.B., Kinlough, C.L., and Kleyman, T.R. (2004). Distinct <span>Pools</span> of <span>Epithelial Sodium Channels Are Expressed</span> at the <span>Plasma Membrane</span>. J. Biol. Chem. <em>279</em>, 48491–48494. <a href="https://doi.org/10.1074/jbc.C400460200">10.1074/jbc.C400460200</a>.</div>
</div>
<div id="ref-reeves2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Reeves, P.J., Callewaert, N., Contreras, R., and Khorana, H.G. (2002). Structure and function in rhodopsin: <span class="nocase">High-level</span> expression of rhodopsin with restricted and homogeneous <span class="nocase">N-glycosylation</span> by a tetracycline-inducible <span class="nocase">N-acetylglucosaminyltransferase I-negative HEK293S</span> stable mammalian cell line. Proceedings of the National Academy of Sciences <em>99</em>, 13419–13424. <a href="https://doi.org/10.1073/pnas.212519299">10.1073/pnas.212519299</a>.</div>
</div>
<div id="ref-shi2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Shi, X., and Jarvis, D.L. (2007). Protein <span class="nocase">N-glycosylation</span> in the baculovirus-insect cell system. Curr Drug Targets <em>8</em>, 1116–1125. <a href="https://doi.org/10.2174/138945007782151360">10.2174/138945007782151360</a>.</div>
</div>
<div id="ref-kashlan11" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Kashlan, O.B., and Kleyman, T.R. (2011). <span>ENaC</span> structure and function in the wake of a resolved structure of a family member. American Journal of Physiology-Renal Physiology <em>301</em>, F684–F696. <a href="https://doi.org/10.1152/ajprenal.00259.2011">10.1152/ajprenal.00259.2011</a>.</div>
</div>
<div id="ref-bruns07" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Bruns, J.B., Carattino, M.D., Sheng, S., Maarouf, A.B., Weisz, O.A., Pilewski, J.M., Hughey, R.P., and Kleyman, T.R. (2007). Epithelial <span>Na</span>+ <span>Channels Are Fully Activated</span> by <span>Furin-</span> and <span class="nocase">Prostasin-dependent Release</span> of an <span>Inhibitory Peptide</span> from the γ-<span>Subunit</span>. J. Biol. Chem. <em>282</em>, 6153–6160. <a href="https://doi.org/10.1074/jbc.M610636200">10.1074/jbc.M610636200</a>.</div>
</div>
<div id="ref-chalfant1999" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Chalfant, M.L., Denton, J.S., Berdiev, B.K., Ismailov, I.I., Benos, D.J., and Stanton, B.A. (1999). Intracellular <span>H</span>+ regulates the α-subunit of <span>ENaC</span>, the epithelial <span>Na</span>+ channel. American Journal of Physiology-Cell Physiology <em>276</em>, C477–C486. <a href="https://doi.org/10.1152/ajpcell.1999.276.2.C477">10.1152/ajpcell.1999.276.2.C477</a>.</div>
</div>
<div id="ref-fejestoth08" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Fejes-Tóth, G., Frindt, G., Náray-Fejes-Tóth, A., and Palmer, L.G. (2008). Epithelial <span>Na</span>+ channel activation and processing in mice lacking <span>SGK1</span>. American Journal of Physiology-Renal Physiology <em>294</em>, F1298–F1305. <a href="https://doi.org/10.1152/ajprenal.00579.2007">10.1152/ajprenal.00579.2007</a>.</div>
</div>
<div id="ref-harris2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Harris, M., Garcia-Caballero, A., Stutts, M.J., Firsov, D., and Rossier, B.C. (2008). Preferential <span>Assembly</span> of <span>Epithelial Sodium Channel</span> (<span>ENaC</span>) <span>Subunits</span> in <span>Xenopus Oocytes</span>: <span>ROLE OF FURIN-MEDIATED ENDOGENOUS PROTEOLYSIS</span> *. Journal of Biological Chemistry <em>283</em>, 7455–7463. <a href="https://doi.org/10.1074/jbc.M707399200">10.1074/jbc.M707399200</a>.</div>
</div>
<div id="ref-hughey2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Hughey, R.P., Bruns, J.B., Kinlough, C.L., Harkleroad, K.L., Tong, Q., Carattino, M.D., Johnson, J.P., Stockand, J.D., and Kleyman, T.R. (2004). Epithelial <span>Sodium Channels Are Activated</span> by <span class="nocase">Furin-dependent Proteolysis</span> *. Journal of Biological Chemistry <em>279</em>, 18111–18114. <a href="https://doi.org/10.1074/jbc.C400080200">10.1074/jbc.C400080200</a>.</div>
</div>
<div id="ref-caldwell2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Caldwell, R.A., Boucher, R.C., and Stutts, M.J. (2005). Neutrophil elastase activates near-silent epithelial <span>Na</span>+ channels and increases airway epithelial <span>Na</span>+ transport. American Journal of Physiology-Lung Cellular and Molecular Physiology <em>288</em>, L813–L819. <a href="https://doi.org/10.1152/ajplung.00435.2004">10.1152/ajplung.00435.2004</a>.</div>
</div>
<div id="ref-hughey03" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Hughey, R.P., Mueller, G.M., Bruns, J.B., Kinlough, C.L., Poland, P.A., Harkleroad, K.L., Carattino, M.D., and Kleyman, T.R. (2003). Maturation of the <span>Epithelial Na</span>+ <span>Channel Involves Proteolytic Processing</span> of the α- and γ-<span>Subunits</span>. J. Biol. Chem. <em>278</em>, 37073–37082. <a href="https://doi.org/10.1074/jbc.M307003200">10.1074/jbc.M307003200</a>.</div>
</div>
<div id="ref-taverna2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Taverna, F., Xiong, Z., Brandes, L., Roder, J.C., Salter, M.W., and MacDonald, J.F. (2000). The <span>Lurcher Mutation</span> of an α-<span class="nocase">Amino-3-hydroxy-5-methyl-</span> 4-isoxazolepropionic <span>Acid Receptor Subunit Enhances Potency</span> of <span>Glutamate</span> and <span>Converts</span> an <span>Antagonist</span> to an <span>Agonist</span> *. Journal of Biological Chemistry <em>275</em>, 8475–8479. <a href="https://doi.org/10.1074/jbc.275.12.8475">10.1074/jbc.275.12.8475</a>.</div>
</div>
<div id="ref-snyder2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Snyder, P.M., Bucher, D.B., and Olson, D.R. (2000). Gating <span>Induces</span> a <span>Conformational Change</span> in the <span>Outer Vestibule</span> of <span>Enac</span>. Journal of General Physiology <em>116</em>, 781–790. <a href="https://doi.org/10.1085/jgp.116.6.781">10.1085/jgp.116.6.781</a>.</div>
</div>
<div id="ref-valeriakalienkova2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Valeria Kalienkova, Mowgli Dandamudi, Cristina Paulino, and Timothy Lynagh (2023). Structural basis for excitatory neuropeptide signaling. bioRxiv, 2023.04.29.538817. <a href="https://doi.org/10.1101/2023.04.29.538817">10.1101/2023.04.29.538817</a>.</div>
</div>
<div id="ref-lingueglia1995" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Lingueglia, E., Champigny, G., Lazdunski, M., and Barbry, P. (1995). Cloning of the amiloride-sensitive <span>FMRFamide</span> peptide-gated sodium channel. Nature <em>378</em>, 730–733. <a href="https://doi.org/10.1038/378730a0">10.1038/378730a0</a>.</div>
</div>
<div id="ref-sheng06" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Sheng, S., Carattino, M.D., Bruns, J.B., Hughey, R.P., and Kleyman, T.R. (2006). Furin cleavage activates the epithelial <span>Na</span>+ channel by relieving <span>Na</span>+ self-inhibition. American Journal of Physiology-Renal Physiology <em>290</em>, F1488–F1496. <a href="https://doi.org/10.1152/ajprenal.00439.2005">10.1152/ajprenal.00439.2005</a>.</div>
</div>
<div id="ref-bize07" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Bize, V., and Horisberger, J.-D. (2007). Sodium self-inhibition of human epithelial sodium channel: Selectivity and affinity of the extracellular sodium sensing site. American Journal of Physiology-Renal Physiology <em>293</em>, F1137–F1146. <a href="https://doi.org/10.1152/ajprenal.00100.2007">10.1152/ajprenal.00100.2007</a>.</div>
</div>
<div id="ref-noreng18" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Noreng, S., Bharadwaj, A., Posert, R., Yoshioka, C., and Baconguis, I. (2018). Structure of the human epithelial sodium channel by cryo-electron microscopy. eLife <em>7</em>, e39340. <a href="https://doi.org/10.7554/eLife.39340">10.7554/eLife.39340</a>.</div>
</div>
<div id="ref-sheng2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Sheng, S., Bruns, J.B., and Kleyman, T.R. (2004). Extracellular <span>Histidine Residues Crucial</span> for <span>Na</span>+ <span class="nocase">Self-inhibition</span> of <span>Epithelial Na</span>+ <span>Channels</span> *. Journal of Biological Chemistry <em>279</em>, 9743–9749. <a href="https://doi.org/10.1074/jbc.M311952200">10.1074/jbc.M311952200</a>.</div>
</div>
</div>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/results/results_transmembrane-domain.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">The Transmembrane Domain Remains Elusive</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/conclusion.html" class="pagination-link">
        <span class="nav-page-text">Conclusion</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"descPosition":"bottom","openEffect":"zoom","selector":".lightbox","closeEffect":"zoom","loop":true});</script>



</body></html>