<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.340">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>ENaC isn’t ASIC - 3&nbsp; ECD Stability Under Activating Conditions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/references.html" rel="next">
<link href="../../parts/results/results_transmembrane-domain.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


</head>

<body class="nav-sidebar floating nav-fixed slimcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../figures/other/domain-cartoon_ECD.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">ENaC isn’t ASIC</span>
    </a>
  </div>
        <div class="quarto-navbar-tools ms-auto">
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.pdf">
              <i class="bi bi-bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="../../posert-dissertation.docx">
              <i class="bi bi-bi-file-word pe-1"></i>
            Download Docx
            </a>
          </li>
      </ul>
    </div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Results</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_activating-conditions.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Frontmatter</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_enac-in-the-body.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">ENaC Biology</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Results</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_transmembrane-domain.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">The Transmembrane Domain Remains Elusive</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_activating-conditions.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/mat-meth.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Materials and Methods</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/processing-flow.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Image Processing</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/validation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Map Validation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/appendices/code.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Code and Computation</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#ecd-conformation-is-unchanged-by-trypsin-cleavage" id="toc-ecd-conformation-is-unchanged-by-trypsin-cleavage" class="nav-link active" data-scroll-target="#ecd-conformation-is-unchanged-by-trypsin-cleavage"><span class="header-section-number">3.1</span> ECD conformation is unchanged by trypsin cleavage</a>
  <ul class="collapse">
  <li><a href="#a-strained-tyrosine-in-the-thumb" id="toc-a-strained-tyrosine-in-the-thumb" class="nav-link" data-scroll-target="#a-strained-tyrosine-in-the-thumb"><span class="header-section-number">3.1.1</span> A strained tyrosine in the thumb</a></li>
  </ul></li>
  <li><a href="#sodium-self-inhibition-does-not-rearrange-the-ecd" id="toc-sodium-self-inhibition-does-not-rearrange-the-ecd" class="nav-link" data-scroll-target="#sodium-self-inhibition-does-not-rearrange-the-ecd"><span class="header-section-number">3.2</span> Sodium self-inhibition does not rearrange the ECD</a></li>
  <li><a href="#glycans-make-important-contacts-among-subunits-and-with-the-membrane" id="toc-glycans-make-important-contacts-among-subunits-and-with-the-membrane" class="nav-link" data-scroll-target="#glycans-make-important-contacts-among-subunits-and-with-the-membrane"><span class="header-section-number">3.3</span> Glycans make important contacts among subunits and with the membrane</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">ECD Stability Under Activating Conditions</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>Despite the fact that the TMD remains elusive, my hope was that investigating conditions thought to open the channel I might observe large-scale ECD rearrangements like those observed in ASIC. In this chapter I cover my efforts on that front, starting with a map of mouse ENaC solved pre- and post-trypsin treatment. Next, I investigate a putative sodium self inhibition site that had been described previously in the literature (including by our own lab) in the presence of Na<sup>+</sup> and K<sup>+</sup>. Finally, I highlight the extensive surface glycosylation I am able to resolve in all of my ECD maps.</p>
<section id="ecd-conformation-is-unchanged-by-trypsin-cleavage" class="level2 page-columns page-full" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="ecd-conformation-is-unchanged-by-trypsin-cleavage"><span class="header-section-number">3.1</span> ECD conformation is unchanged by trypsin cleavage</h2>
<div class="page-columns page-full"><p>ENaC can reliably be opened by trypsin treatment<span class="citation" data-cites="caldwell2004"><sup><a href="../references.html#ref-caldwell2004" role="doc-biblioref">1</a></sup></span>. Additionally, the inhibitory peptide is easily resolved in all of my ECD maps. I wondered whether treatment with trypsin would result in movement of the finger and thumb domains as the model of ASIC-like gating movements predicts. It’s important to note here that I had significant help from Arpita Bharadwaj, who purified and did initial characterization of the mouse material for these grids. Both uncleaved and trypsin-treated mouse ENaC were of similar map quality (3.24 and 3.30 Å respectively with whole-protein masks).</p><div class="no-row-height column-margin column-container"><span class="">Recall that when protons bind ASIC’s acidic pocket, the finger and thumb move approximately 8&nbsp;Å closer together<span class="citation" data-cites="yoder2018"><sup><a href="../references.html#ref-yoder2018" role="doc-biblioref">2</a></sup></span>.</span></div></div>
<!-- TODO: split up this figure? -->
<p>Biochemical analyses show that trypsin treatment cleaves both the α and γ subunits (<a href="#fig-trypsin-treatment">Figure&nbsp;<span>3.1</span></a> A, B). Additionally, trypsin treatment of <em>X. laevis</em> oocytes expressing mouse ENaC<sub>αβγ</sub> show the expected increase in amiloride sensitive current (<a href="#fig-trypsin-treatment">Figure&nbsp;<span>3.1</span></a> C). However, aside from the disappearance of the inhibitory peptide, the ECD is otherwise superimposable (RMSD 0.440&nbsp;Å; <a href="#fig-trypsin-treatment">Figure&nbsp;<span>3.1</span></a> D, E). The finger and thumb do not shift even a fraction of what the ASIC-like model would predict. Based on the observed cryoEM map density, only the γ subunit GRIP domain is released from the channel despite the fact that biochemical characterization indicates both are digested by trypsin. I also note that the entire GRIP domain, not just the inhibitory peptide, has reduced map density in the trypsin-treated sample. This suggests that the entire grip domain might be released from the subunit, but of course it’s equally possible that release of the inhibitory peptide results in increased GRIP flexibility, which blurs out the map.</p>
<div id="fig-trypsin-treatment" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/trypsin-western/trypsin-western.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Trypsin cleavage does not rearrange the ECD. A: Western blot of αENaC and γENaC. Samples were treated with trypsin (10&nbsp;μg/mL for 5 minutes) and aprotinin (10 mg/mL) as indicated. Bands corresponding to expected uncleaved αENaC and γENaC are marked with an asterisk. B: Silver-stained SDS-PAGE gel of mouse ENaC treated with trypsin indicates cleavage of all subunits. C: Electrophysiological recordings of mouse ENaC expressed in X. laevis oocytes. D and E: cryoEM maps of the γ and α subunits, respectively. The inhibitory peptide is colored in each panel, corresponding to the uncleaved (yellow, top) or cleaved (purple, bottom) map. A cartoon representation of the ECD is presented in the center to orient the reader, with the inhibitory peptide displayed as spheres."><img src="figures/trypsin-western/trypsin-western.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.1: Trypsin cleavage does not rearrange the ECD. <strong>A</strong>: Western blot of αENaC and γENaC. Samples were treated with trypsin (10&nbsp;μg/mL for 5 minutes) and aprotinin (10 mg/mL) as indicated. Bands corresponding to expected uncleaved αENaC and γENaC are marked with an asterisk. <strong>B</strong>: Silver-stained SDS-PAGE gel of mouse ENaC treated with trypsin indicates cleavage of all subunits. <strong>C</strong>: Electrophysiological recordings of mouse ENaC expressed in <em>X. laevis</em> oocytes. <strong>D</strong> and <strong>E</strong>: cryoEM maps of the γ and α subunits, respectively. The inhibitory peptide is colored in each panel, corresponding to the uncleaved (yellow, top) or cleaved (purple, bottom) map. A cartoon representation of the ECD is presented in the center to orient the reader, with the inhibitory peptide displayed as spheres.</figcaption>
</figure>
</div>
<p>These results are incompatible with a large-scale movement of the ECD upon trypsin digestion. The existence of some γ GRIP density indicates that the trypsin-treated map is an ensemble of particles which have and have not released the γ GRIP domain. However, I do not observe any blurring of the γ thumb, which would be expected in a map comprising particles with significantly shifted conformations. I must therefore conclude that the removal of the γ inhibitory domain is not sufficient, on its own, to induce such changes. Additionally, I note that these maps are the first indication that trypsin processing of the α subunit does not release its GRIP domain. This is surprising but perhaps not wholly unexpected; the γ subunit is known to be essential for activation by trypsin but the same has not yet been shown for αENaC<span class="citation" data-cites="diakov2008"><sup><a href="../references.html#ref-diakov2008" role="doc-biblioref">3</a></sup></span>.</p>
<section id="a-strained-tyrosine-in-the-thumb" class="level3" data-number="3.1.1">
<h3 data-number="3.1.1" class="anchored" data-anchor-id="a-strained-tyrosine-in-the-thumb"><span class="header-section-number">3.1.1</span> A strained tyrosine in the thumb</h3>
<div id="fig-tyr-rotamer" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/tyr-rotamer/all-tyr-figure.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="A strained tyrosine rotamer is present in the γ subunit is released by trypsin cleavage. A: Human CKO γY425 (right), with approximate position indicated by a star in the cartoon (left). Hydrogen bonds are colored blue for ideal geometry and orange for slightly strained distance. B: γY425 (Y431 in mouse) colored by construct: human CKO, human CKO/DEG, mouse uncleaved, and mouse trypsin as red, blue, yellow, and purple respectively. C: αY447 (Y474 in mouse) colored by construct. D: Comparison of the inhibitory peptides of α (blue), β (red), and γ (magenta). The displayed model is human CKO digitonin, but the peptides are superimposable in all six models."><img src="figures/tyr-rotamer/all-tyr-figure.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.2: A strained tyrosine rotamer is present in the γ subunit is released by trypsin cleavage. <strong>A</strong>: Human CKO γY425 (right), with approximate position indicated by a star in the cartoon (left). Hydrogen bonds are colored blue for ideal geometry and orange for slightly strained distance. <strong>B</strong>: γY425 (Y431 in mouse) colored by construct: human CKO, human CKO/DEG, mouse uncleaved, and mouse trypsin as red, blue, yellow, and purple respectively. <strong>C</strong>: αY447 (Y474 in mouse) colored by construct. <strong>D</strong>: Comparison of the inhibitory peptides of α (blue), β (red), and γ (magenta). The displayed model is human CKO digitonin, but the peptides are superimposable in all six models.</figcaption>
</figure>
</div>
<div id="fig-tyr-rotamer-maps" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/tyr-rotamer/supplement-maps.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="CryoEM map potential for strained γENaC tyrosine for all maps/models. The maps are masked around the residue of interest."><img src="figures/tyr-rotamer/supplement-maps.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.3: CryoEM map potential for strained γENaC tyrosine for all maps/models. The maps are masked around the residue of interest.</figcaption>
</figure>
</div>
<p>Our high-resolution maps of the γ ECD reveal an interesting rotamer outlier, γY425 (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>3.2</span></a>). The map for this residue is of high quality in five of the six maps, supporting its unlikely (prior probability of 0.035%) conformation (fig-tyr-rotamer-maps). The single map with poor quality is the trypsin-treated mouse map, in which the density is blurred and unclear. Human γY425 forms hydrogen bonds with the finger (H233) and inhibitory peptide (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>3.2</span></a> A). This tyrosine adopts the same pose in all models <em>except</em> the trypsin-treated mouse model, in which it may rotate to point toward the outer GRIP domain (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>3.2</span></a> B). Uniquely among our maps, density for γY431 (the equivalent position to human Y425) is poor in the mouse trypsin map, so I cannot assign this rotamer with certainty. The equivalent residue in α is also a tyrosine (αY447 and 474 in human and mouse respectively), but the inhibitory peptide curves away from the channel center, removing the interactions with the backbone and aromatic oxygen atoms and allowing a more favored rotamer (<a href="#fig-tyr-rotamer">Figure&nbsp;<span>3.2</span></a> C and D). The same position in β, N417, only forms a hydrogen bond with the finger domain (N225). The lack of any interactions between βN417 and the βENaC GRIP domain, which is not cleaved to open the channel, is further suggestive of this residue’s role in proteolytic gating. I note also that the map quality worsens for the nearby γM438, mutation of which alters channel gating<span class="citation" data-cites="maarouf2009"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a></sup></span>. The reduced quality for this map region upon trypsin treatment may represent increased flexibility, perhaps indicating that conformational strain in the upper thumb is released along with the inhibitory peptide.</p>
</section>
</section>
<section id="sodium-self-inhibition-does-not-rearrange-the-ecd" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="sodium-self-inhibition-does-not-rearrange-the-ecd"><span class="header-section-number">3.2</span> Sodium self-inhibition does not rearrange the ECD</h2>
<p>Uncleaved ENaC has a high P<sub>O</sub> at low external Na<sup>+</sup> concentrations and a low P<sub>O</sub> at high external Na<sup>+</sup>; this attribute is called sodium self-inhibition (SSI)<span class="citation" data-cites="sheng06"><sup><a href="../references.html#ref-sheng06" role="doc-biblioref">5</a></sup></span>. High extracellular K<sup>+</sup> does not prime inhibition in wild-type channels and SSI is rapidly attenuated upon exposure to low (i.e., &lt;10&nbsp;mM) Na<sup>+</sup><span class="citation" data-cites="maarouf2009 kashlan2015"><sup><a href="../references.html#ref-maarouf2009" role="doc-biblioref">4</a>,<a href="../references.html#ref-kashlan2015" role="doc-biblioref">6</a></sup></span>. The linker connecting β6 and β7 is important for SSI, and I have previously reported evidence of a putative bound Na<sup>+</sup> ion in this region<span class="citation" data-cites="kashlan2015 noreng20"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">6</a>,<a href="../references.html#ref-noreng20" role="doc-biblioref">7</a></sup></span>. Moreover, cross-linking of several pairs of amino acids in the ECD affects SSI magnitude, implying that some rearrangement is required for SSI<span class="citation" data-cites="kashlan2015 zhang2023"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">6</a>,<a href="../references.html#ref-zhang2023" role="doc-biblioref">8</a></sup></span>. Given this body of research, I solved structures of ENaC in high (200&nbsp;mM) and low (nominal 0&nbsp;mM&nbsp;Na<sup>+</sup>, 200&nbsp;mM&nbsp;K<sup>+</sup>) sodium. I expected the high-Na<sup>+</sup> model to show rearrangements throughout the ECD, as would be required to transfer Na<sup>+</sup>-binding state to the pore.</p>
<p>Our maps of human CKO ENaC in Na<sup>+</sup> and K<sup>+</sup> buffers achieved resolutions of 2.3 and 3.0&nbsp;Å, respectively. The Na<sup>+</sup> structure was solved in digitonin with both the α- and β-binding Fabs, while the K<sup>+</sup> structure was solved in nanodiscs without any Fabs. Although previously it has proven difficult to break the pseudo-threefold symmetry without including Fabs, I worried that their inclusion may affect SSI. Fortunately, the high quality of our cryoEM data allowed us to break the symmetry with neither Fabs nor an external reference — I believe the major symmetry breaking features are micelle-proximal glycans (Supplemental <a href="../appendices/mat-meth.html#fig-fab-compare">Figure&nbsp;<span>A.1</span></a>). Binding of either Fab induces minimal rearrangement of ENaC; the β epitope does not change conformation upon binding, while the α epitope pulls the loop from αP264-T274 approximately 4.5&nbsp;Å away from the protein core without distorting the structure outside that loop.</p>
<div id="fig-sodium-self-inhib" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/sodium-self-inhibition/ssi.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Presence or absence of Na+ does not rearrange the ECD. A: A previously identified ion-binding site still shows cryoEM map density with both Na+ (left) and K+ (right). The green dot corresponds to the previously-reported (PDB 6WTH) putative ion position. B: The β6-β7 linker in Na+ or K+ buffers."><img src="figures/sodium-self-inhibition/ssi.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.4: Presence or absence of Na<sup>+</sup> does not rearrange the ECD. <strong>A</strong>: A previously identified ion-binding site still shows cryoEM map density with both Na<sup>+</sup> (left) and K<sup>+</sup> (right). The green dot corresponds to the previously-reported (PDB 6WTH) putative ion position. <strong>B</strong>: The β6-β7 linker in Na<sup>+</sup> or K<sup>+</sup> buffers.</figcaption>
</figure>
</div>
<p>To our surprise, no resolved region of the ECD undergoes any significant structural rearrangement between high- and low-Na<sup>+</sup> buffers (RMSD 0.490&nbsp;Å, <a href="#fig-sodium-self-inhib">Figure&nbsp;<span>3.4</span></a> A). Aside from the changes induced by the α-binding Fab, the only Cα shifts greater than 2 Å are residues terminating the resolved regions of the channel, which are likely a result of poor map quality. An acidic cleft in the αENaC finger, near the β-ball, is implicated in sodium self-inhibition<span class="citation" data-cites="kashlan2015"><sup><a href="../references.html#ref-kashlan2015" role="doc-biblioref">6</a></sup></span>. However, our cryoEM maps have density for a ligand in the acidic cleft in both the Na<sup>+</sup> and K<sup>+</sup> maps (<a href="#fig-sodium-self-inhib">Figure&nbsp;<span>3.4</span></a> A), although the site is slightly expanded in the K<sup>+</sup> map, perhaps to accommodate the larger K<sup>+</sup> ion. The β6-β7 linker has high quality density in both maps, supporting an almost-identical (Cα RMSD 1.076&nbsp;Å when the whole α subunit is aligned by the palm domain) conformation, aside from the blurring of αR333 across multiple rotamers (<a href="#fig-sodium-self-inhib">Figure&nbsp;<span>3.4</span></a> B). Although SSI is sensitive to pH, the buffer conditions for the CKO map should be at least 55% inhibited<span class="citation" data-cites="collier2009"><sup><a href="../references.html#ref-collier2009" role="doc-biblioref">9</a></sup></span>. Attempts at detecting other classes of particle in the CKO ENaC (Na<sup>+</sup>) map were not successful, leading us to believe that the particles are homogeneous. The lack of any significant structural difference in the acidic pocket in the presence of Na<sup>+</sup> and K<sup>+</sup> is more compatible with an SSI model in which Na<sup>+</sup>-bound ENaC is “locked” in a closed state, rather than some third inhibited state.</p>
</section>
<section id="glycans-make-important-contacts-among-subunits-and-with-the-membrane" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="glycans-make-important-contacts-among-subunits-and-with-the-membrane"><span class="header-section-number">3.3</span> Glycans make important contacts among subunits and with the membrane</h2>
<p>Glycosylation of ENaC at conserved asparagine sites is essential for sodium self inhibition, trypsin activation, and mechanosensitivity<span class="citation" data-cites="kashlan2018 knoepp2020"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">10</a>,<a href="../references.html#ref-knoepp2020" role="doc-biblioref">11</a></sup></span>. The high-quality maps obtained in this study enable modeling of significantly more and longer glycans than in previous work. I find a total of 19 distinct glycosylation sites, with at most 9 sugars I are able to build into a single glycan. For the most part, the sugars are identical in all six models and make no protein or glycan interactions. These sugars likely play a role in other regulatory roles, such as channel trafficking and maturation or interaction with the extracellular matrix<span class="citation" data-cites="kashlan2018 hughey04 knoepp2020"><sup><a href="../references.html#ref-kashlan2018" role="doc-biblioref">10</a>–<a href="../references.html#ref-hughey04" role="doc-biblioref">12</a></sup></span>. However, two classes of glycan are particularly interesting: the micelle-proximal and the intersubunit glycans.</p>
<div id="fig-intersub-glyc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/intersubunit-glyc/intersub-glyc.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="ENaC’s glycosylation sites form important intersubunit interactions. A: Cartoon overview of intersubunit glycosylations. Dotted lines represent glycan chains. Hexagons represent sugars, circles represent potentially-interacting residues. Unfiltered maps for the first two sugars are presented to indicate confidence in Asn assignment. B: Silver-stained SDS-PAGE gel of protein treated with PNGase F. A shift indicates that the purified protein is glycosylated. C: Intersubunit glycan from βN260 to αD235, K495. D: Intersubunit interaction between glycans from βN207 (red) and γN271 (magenta). In both C and D, the solid surface is a gaussian-blurred version of the sharp human CKO digitonin map (mesh)."><img src="figures/intersubunit-glyc/intersub-glyc.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.5: ENaC’s glycosylation sites form important intersubunit interactions. <strong>A</strong>: Cartoon overview of intersubunit glycosylations. Dotted lines represent glycan chains. Hexagons represent sugars, circles represent potentially-interacting residues. Unfiltered maps for the first two sugars are presented to indicate confidence in Asn assignment. <strong>B</strong>: Silver-stained SDS-PAGE gel of protein treated with PNGase F. A shift indicates that the purified protein is glycosylated. <strong>C</strong>: Intersubunit glycan from βN260 to αD235, K495. <strong>D</strong>: Intersubunit interaction between glycans from βN207 (red) and γN271 (magenta). In both C and D, the solid surface is a gaussian-blurred version of the sharp human CKO digitonin map (mesh).</figcaption>
</figure>
</div>
<p>The intersubunit glycans stretch from one subunit to another (<a href="#fig-intersub-glyc">Figure&nbsp;<span>3.5</span></a> A). I are able to model long chains for these glycans, with some maps supporting up to nine monosaccharides. Due to the flexibility of these long chains, the terminal side is quite blurred out in our cryoEM maps, and so I cannot be confident about the exact orientation or identity of the constituent sugars. Nonetheless, our maps are of sufficient quality to at least suggest sugar-mediated interactions between subunits, and the first sugars are of sufficient resolution to assign the asparagine residues to which they are coupled. Treatment with PNGase F induces a size shift in our purified protein, confirming the identity of these densities as glycans (<a href="#fig-intersub-glyc">Figure&nbsp;<span>3.5</span></a> B) I built typical high-mannose glycans (Man<sub>5</sub>GlcNAc<sub>2</sub>) into our models as expected for GnTI<sup>-</sup> and Sf9 cells to assess these interactions at a low resolution<span class="citation" data-cites="reeves2002 shi2007"><sup><a href="../references.html#ref-reeves2002" role="doc-biblioref">13</a>,<a href="../references.html#ref-shi2007" role="doc-biblioref">14</a></sup></span>. One glycan reaches from βN260 to the α subunit (<a href="#fig-intersub-glyc">Figure&nbsp;<span>3.5</span></a> C), potentially hydrogen bonding with αD235 and αK459. These three residues are absolutely conserved across ENaC genes surveyed. Similarly, glycans attached to conserved sites βN207 and γN271 connect the β GRIP to the γ palm (<a href="#fig-intersub-glyc">Figure&nbsp;<span>3.5</span></a> D). Knock-out of sugars in similar positions in αENaC increase blood pressure in mice<span class="citation" data-cites="knoepp2020"><sup><a href="../references.html#ref-knoepp2020" role="doc-biblioref">11</a></sup></span>. The relatively high quality of these glycans suggest they are quite conformationally stable and may play a role in channel function.</p>
<div id="fig-micelle-glyc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><a href="figures/membrane-glyc/membrane-glyc.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="ENaC’s glycans contact the micelle. A: An overview of the position of the micelle-proximal glycans and the micelle. The image on the right is a zoomed view of the boxed region on the left B: In-grid rotation distribution for two selected classes of CKO nanodiscs. Note that both classes have even distribution of in-plane angles, indicating that the micelle tilt does not align with some grid force such as fluid flow during blotting. C: A cartoon representation of the position of the membrane-proximal glycans relative to the micelle tilt axis. D and E: The sharpened map of human CKO digitonin for the first sugar attached to αN397 and βN364, respectively."><img src="figures/membrane-glyc/membrane-glyc.png" class="img-fluid figure-img"></a></p>
<figcaption class="figure-caption">Figure&nbsp;3.6: ENaC’s glycans contact the micelle. <strong>A</strong>: An overview of the position of the micelle-proximal glycans and the micelle. The image on the right is a zoomed view of the boxed region on the left <strong>B</strong>: In-grid rotation distribution for two selected classes of CKO nanodiscs. Note that both classes have even distribution of in-plane angles, indicating that the micelle tilt does not align with some grid force such as fluid flow during blotting. <strong>C</strong>: A cartoon representation of the position of the membrane-proximal glycans relative to the micelle tilt axis. <strong>D and E</strong>: The sharpened map of human CKO digitonin for the first sugar attached to αN397 and βN364, respectively.</figcaption>
</figure>
</div>
<p>The micelle-proximal glycans are anchored to hαN397 and hβN364 and the equivalent mouse residues, mαN424 and mβN362 (<a href="#fig-micelle-glyc">Figure&nbsp;<span>3.6</span></a> A). The α site is conserved in mammalian ENaCs, while the β site is conserved in all surveyed organisms except lungfish. These glycans extend away from the ECD body and turn down toward the micelle or lipids and are clear in all ENaC maps solved to date. The γ subunit does not have an equivalent asparagine residue, and so does not have a micelle-proximal glycan. I note that all ENaC maps produced to date also show a consistent tilt of the micelle or nanodisc such that the γ subunit is closer to the surface of the micelle than are the α or β subunits; this tilt does not appear to be an artifact of in-plane rotation (<a href="#fig-micelle-glyc">Figure&nbsp;<span>3.6</span></a> B). I propose therefore that this tilt may in part be caused by the presence of the micelle-proximal glycans, which are only missing from γENaC (<a href="#fig-micelle-glyc">Figure&nbsp;<span>3.6</span></a> C). Their presence in the nanodisc map may indicate that they perform a similar role in a native cell membrane context.</p>
<!-- Move this to the end of content so the margins are right in citations -->


<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-caldwell2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Caldwell, R.A., Boucher, R.C., and Stutts, M.J. (2004). Serine protease activation of near-silent epithelial <span>Na</span>+ channels. American Journal of Physiology-Cell Physiology <em>286</em>, C190–C194. <a href="https://doi.org/10.1152/ajpcell.00342.2003">10.1152/ajpcell.00342.2003</a>.</div>
</div>
<div id="ref-yoder2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Yoder, N., Yoshioka, C., and Gouaux, E. (2018). Gating mechanisms of acid-sensing ion channels. Nature <em>555</em>, 397–401. <a href="https://doi.org/10.1038/nature25782">10.1038/nature25782</a>.</div>
</div>
<div id="ref-diakov2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Diakov, A., Bera, K., Mokrushina, M., Krueger, B., and Korbmacher, C. (2008). Cleavage in the γ-subunit of the epithelial sodium channel (<span>ENaC</span>) plays an important role in the proteolytic activation of near-silent channels. J Physiol <em>586</em>, 4587–4608. <a href="https://doi.org/10.1113/jphysiol.2008.154435">10.1113/jphysiol.2008.154435</a>.</div>
</div>
<div id="ref-maarouf2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Maarouf, A.B., Sheng, N., Chen, J., Winarski, K.L., Okumura, S., Carattino, M.D., Boyd, C.R., Kleyman, T.R., and Sheng, S. (2009). Novel <span>Determinants</span> of <span>Epithelial Sodium Channel Gating</span> within <span>Extracellular Thumb Domains</span> *. Journal of Biological Chemistry <em>284</em>, 7756–7765. <a href="https://doi.org/10.1074/jbc.M807060200">10.1074/jbc.M807060200</a>.</div>
</div>
<div id="ref-sheng06" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Sheng, S., Carattino, M.D., Bruns, J.B., Hughey, R.P., and Kleyman, T.R. (2006). Furin cleavage activates the epithelial <span>Na</span>+ channel by relieving <span>Na</span>+ self-inhibition. American Journal of Physiology-Renal Physiology <em>290</em>, F1488–F1496. <a href="https://doi.org/10.1152/ajprenal.00439.2005">10.1152/ajprenal.00439.2005</a>.</div>
</div>
<div id="ref-kashlan2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Kashlan, O.B., Blobner, B.M., Zuzek, Z., Tolino, M., and Kleyman, T.R. (2015). Na+ <span>Inhibits</span> the <span>Epithelial Na</span>+ <span>Channel</span> by <span>Binding</span> to a <span>Site</span> in an <span>Extracellular Acidic Cleft</span> *. Journal of Biological Chemistry <em>290</em>, 568–576. <a href="https://doi.org/10.1074/jbc.M114.606152">10.1074/jbc.M114.606152</a>.</div>
</div>
<div id="ref-noreng20" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Noreng, S., Posert, R., Bharadwaj, A., Houser, A., and Baconguis, I. (2020). Molecular principles of assembly, activation, and inhibition in epithelial sodium channel. eLife <em>9</em>, e59038. <a href="https://doi.org/10.7554/eLife.59038">10.7554/eLife.59038</a>.</div>
</div>
<div id="ref-zhang2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Zhang, L., Wang, X., Chen, J., Sheng, S., and Kleyman, T.R. (2023). Extracellular intersubunit interactions modulate epithelial <span>Na</span>+ channel gating. Journal of Biological Chemistry <em>299</em>, 102914. <a href="https://doi.org/10.1016/j.jbc.2023.102914">10.1016/j.jbc.2023.102914</a>.</div>
</div>
<div id="ref-collier2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Collier, D.M., and Snyder, P.M. (2009). Extracellular <span>Protons Regulate Human ENaC</span> by <span>Modulating Na</span>+ <span class="nocase">Self-inhibition</span> *. Journal of Biological Chemistry <em>284</em>, 792–798. <a href="https://doi.org/10.1074/jbc.M806954200">10.1074/jbc.M806954200</a>.</div>
</div>
<div id="ref-kashlan2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Kashlan, O.B., Kinlough, C.L., Myerburg, M.M., Shi, S., Chen, J., Blobner, B.M., Buck, T.M., Brodsky, J.L., Hughey, R.P., and Kleyman, T.R. (2018). N-linked glycans are required on epithelial <span>Na</span>(+) channel subunits for maturation and surface expression. Am. J. Physiol. Renal Physiol. <em>314</em>, F483–f492. <a href="https://doi.org/10.1152/ajprenal.00195.2017">10.1152/ajprenal.00195.2017</a>.</div>
</div>
<div id="ref-knoepp2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Knoepp, F., Ashley, Z., Barth, D., Baldin, J.-P., Jennings, M., Kazantseva, M., Saw, E.L., Katare, R., Alvarez de la Rosa, D., Weissmann, N., et al. (2020). Shear force sensing of epithelial <span>Na</span>+ channel (<span>ENaC</span>) relies on <span class="nocase">N-glycosylated</span> asparagines in the palm and knuckle domains of <span class="nocase">αENaC</span>. Proceedings of the National Academy of Sciences <em>117</em>, 717–726. <a href="https://doi.org/10.1073/pnas.1911243117">10.1073/pnas.1911243117</a>.</div>
</div>
<div id="ref-hughey04" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Hughey, R.P., Bruns, J.B., Kinlough, C.L., and Kleyman, T.R. (2004). Distinct <span>Pools</span> of <span>Epithelial Sodium Channels Are Expressed</span> at the <span>Plasma Membrane</span>. J. Biol. Chem. <em>279</em>, 48491–48494. <a href="https://doi.org/10.1074/jbc.C400460200">10.1074/jbc.C400460200</a>.</div>
</div>
<div id="ref-reeves2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Reeves, P.J., Callewaert, N., Contreras, R., and Khorana, H.G. (2002). Structure and function in rhodopsin: <span class="nocase">High-level</span> expression of rhodopsin with restricted and homogeneous <span class="nocase">N-glycosylation</span> by a tetracycline-inducible <span class="nocase">N-acetylglucosaminyltransferase I-negative HEK293S</span> stable mammalian cell line. Proceedings of the National Academy of Sciences <em>99</em>, 13419–13424. <a href="https://doi.org/10.1073/pnas.212519299">10.1073/pnas.212519299</a>.</div>
</div>
<div id="ref-shi2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Shi, X., and Jarvis, D.L. (2007). Protein <span class="nocase">N-glycosylation</span> in the baculovirus-insect cell system. Curr Drug Targets <em>8</em>, 1116–1125. <a href="https://doi.org/10.2174/138945007782151360">10.2174/138945007782151360</a>.</div>
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/results/results_transmembrane-domain.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">The Transmembrane Domain Remains Elusive</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/references.html" class="pagination-link">
        <span class="nav-page-text">References</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","selector":".lightbox","descPosition":"bottom","loop":true,"openEffect":"zoom"});</script>



</body></html>