# ECD Stability Under Activating Conditions

Despite the fact that the TMD remains elusive, my hope was that investigating conditions thought to open the channel I might observe large-scale ECD rearrangements like those observed in ASIC.
In this chapter I cover my efforts on that front, starting with a map of mouse ENaC solved pre- and post-trypsin treatment.
Next, I investigate a putative sodium self inhibition site that had been described previously in the literature (including by our own lab) in the presence of Na^+^ and K^+^.
Finally, I highlight the extensive surface glycosylation I am able to resolve in all of my ECD maps.

## ECD conformation is unchanged by trypsin cleavage

ENaC can reliably be opened by trypsin treatment[@caldwell2004].
Additionally, the inhibitory peptide is easily resolved in all of my ECD maps.
I wondered whether treatment with trypsin would result in movement of the finger and thumb domains as the model of ASIC-like gating movements predicts.[Recall that when protons bind ASIC's acidic pocket, the finger and thumb move approximately 8&nbsp;&Aring; closer together [@yoder2018].]{.aside}
It's important to note here that I had significant help from Arpita Bharadwaj, who purified and did initial characterization of the mouse material for these grids.
Both uncleaved and trypsin-treated mouse ENaC were of similar map quality (3.24 and 3.30 &Aring; respectively with whole-protein masks).

<!-- TODO: split up this figure? -->

Biochemical analyses show that trypsin treatment cleaves both the &alpha; and &gamma; subunits (@fig-trypsin-treatment A, B).
Additionally, trypsin treatment of *X. laevis* oocytes expressing mouse ENaC~&alpha;&beta;&gamma;~ show the expected increase in amiloride sensitive current (@fig-trypsin-treatment C).
However, aside from the disappearance of the inhibitory peptide, the ECD is otherwise superimposable (RMSD 0.440&nbsp;&Aring;; @fig-trypsin-treatment D, E).
The finger and thumb do not shift even a fraction of what the ASIC-like model would predict.
Based on the observed cryoEM map density, only the &gamma; subunit GRIP domain is released from the channel despite the fact that biochemical characterization indicates both are digested by trypsin.
I also note that the entire GRIP domain, not just the inhibitory peptide, has reduced map density in the trypsin-treated sample.
This suggests that the entire grip domain might be released from the subunit, but of course it's equally possible that release of the inhibitory peptide results in increased GRIP flexibility, which blurs out the map.

![Trypsin cleavage does not rearrange the ECD. **A**: Western blot of &alpha;ENaC and &gamma;ENaC. Samples were treated with trypsin (10&nbsp;&mu;g/mL for 5 minutes) and aprotinin (10 mg/mL) as indicated. Bands corresponding to expected uncleaved &alpha;ENaC and &gamma;ENaC are marked with an asterisk. **B**: Silver-stained SDS-PAGE gel of mouse ENaC treated with trypsin indicates cleavage of all subunits. **C**: Electrophysiological recordings of mouse ENaC expressed in *X. laevis* oocytes. **D** and **E**: cryoEM maps of the &gamma; and &alpha; subunits, respectively. The inhibitory peptide is colored in each panel, corresponding to the uncleaved (yellow, top) or cleaved (purple, bottom) map. A cartoon representation of the ECD is presented in the center to orient the reader, with the inhibitory peptide displayed as spheres.](figures/trypsin-western/trypsin-western.png){#fig-trypsin-treatment}

These results are incompatible with a large-scale movement of the ECD upon trypsin digestion.
The existence of some &gamma; GRIP density indicates that the trypsin-treated map is an ensemble of particles which have and have not released the &gamma; GRIP domain.
However, I do not observe any blurring of the &gamma; thumb, which would be expected in a map comprising particles with significantly shifted conformations.
I must therefore conclude that the removal of the &gamma; inhibitory domain is not sufficient, on its own, to induce such changes.
Additionally, I note that these maps are the first indication that trypsin processing of the &alpha; subunit does not release its GRIP domain.
This is surprising but perhaps not wholly unexpected; the &gamma; subunit is known to be essential for activation by trypsin but the same has not yet been shown for &alpha;ENaC[@diakov2008].

### A strained tyrosine in the thumb

![A strained tyrosine rotamer is present in the &gamma; subunit is released by trypsin cleavage. **A**: Human CKO &gamma;Y425 (right), with approximate position indicated by a star in the cartoon (left). Hydrogen bonds are colored blue for ideal geometry and orange for slightly strained distance. **B**: &gamma;Y425 (Y431 in mouse) colored by construct: human CKO, human CKO/DEG, mouse uncleaved, and mouse trypsin as red, blue, yellow, and purple respectively. **C**: &alpha;Y447 (Y474 in mouse) colored by construct. **D**: Comparison of the inhibitory peptides of &alpha; (blue), &beta; (red), and &gamma; (magenta). The displayed model is human CKO digitonin, but the peptides are superimposable in all six models.](figures/tyr-rotamer/all-tyr-figure.png){#fig-tyr-rotamer}

![CryoEM map potential for strained &gamma;ENaC tyrosine for all maps/models. The maps are masked around the residue of interest.](figures/tyr-rotamer/supplement-maps.png){#fig-tyr-rotamer-maps}

Our high-resolution maps of the &gamma; ECD reveal an interesting rotamer outlier, &gamma;Y425 (@fig-tyr-rotamer).
The map for this residue is of high quality in five of the six maps, supporting its unlikely (prior probability of 0.035%) conformation (fig-tyr-rotamer-maps).
The single map with poor quality is the trypsin-treated mouse map, in which the density is blurred and unclear.
Human &gamma;Y425 forms hydrogen bonds with the finger (H233) and inhibitory peptide (@fig-tyr-rotamer A).
This tyrosine adopts the same pose in all models *except* the trypsin-treated mouse model, in which it may rotate to point toward the outer GRIP domain (@fig-tyr-rotamer B).
Uniquely among our maps, density for &gamma;Y431 (the equivalent position to human Y425) is poor in the mouse trypsin map, so I cannot assign this rotamer with certainty.
The equivalent residue in &alpha; is also a tyrosine (&alpha;Y447 and 474 in human and mouse respectively), but the inhibitory peptide curves away from the channel center, removing the interactions with the backbone and aromatic oxygen atoms and allowing a more favored rotamer (@fig-tyr-rotamer C and D).
The same position in &beta;, N417, only forms a hydrogen bond with the finger domain (N225).
The lack of any interactions between &beta;N417 and the &beta;ENaC GRIP domain, which is not cleaved to open the channel, is further suggestive of this residue's role in proteolytic gating.
I note also that the map quality worsens for the nearby &gamma;M438, mutation of which alters channel gating[@maarouf2009].
The reduced quality for this map region upon trypsin treatment may represent increased flexibility, perhaps indicating that conformational strain in the upper thumb is released along with the inhibitory peptide.

## Sodium self-inhibition does not rearrange the ECD

Uncleaved ENaC has a high P~O~ at low external Na^+^ concentrations and a low P~O~ at high external Na^+^; this attribute is called sodium self-inhibition (SSI)[@sheng06].
High extracellular K^+^ does not prime inhibition in wild-type channels and SSI is rapidly attenuated upon exposure to low (i.e., &lt;10&nbsp;mM) Na^+^[@maarouf2009; @kashlan2015].
The linker connecting &beta;6 and &beta;7 is important for SSI, and I have previously reported evidence of a putative bound Na^+^ ion in this region[@kashlan2015; @noreng20].
Moreover, cross-linking of several pairs of amino acids in the ECD affects SSI magnitude, implying that some rearrangement is required for SSI[@kashlan2015; @zhang2023].
Given this body of research, I solved structures of ENaC in high (200&nbsp;mM) and low (nominal 0&nbsp;mM&nbsp;Na^+^, 200&nbsp;mM&nbsp;K^+^) sodium.
I expected the high-Na^+^ model to show rearrangements throughout the ECD, as would be required to transfer Na^+^-binding state to the pore.

Our maps of human CKO ENaC in Na^+^ and K^+^ buffers achieved resolutions of 2.3 and 3.0&nbsp;&Aring;, respectively.
The Na^+^ structure was solved in digitonin with both the &alpha;- and &beta;-binding Fabs, while the K^+^ structure was solved in nanodiscs without any Fabs.
Although previously it has proven difficult to break the pseudo-threefold symmetry without including Fabs, I worried that their inclusion may affect SSI.
Fortunately, the high quality of our cryoEM data allowed us to break the symmetry with neither Fabs nor an external reference &mdash; I believe the major symmetry breaking features are micelle-proximal glycans (Supplemental @fig-fab-compare).
Binding of either Fab induces minimal rearrangement of ENaC; the &beta; epitope does not change conformation upon binding, while the &alpha; epitope pulls the loop from &alpha;P264-T274 approximately 4.5&nbsp;&Aring; away from the protein core without distorting the structure outside that loop.

![Presence or absence of Na^+^ does not rearrange the ECD. **A**: A previously identified ion-binding site still shows cryoEM map density with both Na^+^ (left) and K^+^ (right). The green dot corresponds to the previously-reported (PDB 6WTH) putative ion position. **B**: The &beta;6-&beta;7 linker in Na^+^ or K^+^ buffers.](figures/sodium-self-inhibition/ssi.png){#fig-sodium-self-inhib}

To our surprise, no resolved region of the ECD undergoes any significant structural rearrangement between high- and low-Na^+^ buffers (RMSD 0.490&nbsp;&Aring;, @fig-sodium-self-inhib A).
Aside from the changes induced by the &alpha;-binding Fab, the only C&alpha; shifts greater than 2 &Aring; are residues terminating the resolved regions of the channel, which are likely a result of poor map quality.
An acidic cleft in the &alpha;ENaC finger, near the &beta;-ball, is implicated in sodium self-inhibition[@kashlan2015].
However, our cryoEM maps have density for a ligand in the acidic cleft in both the Na^+^ and K^+^ maps (@fig-sodium-self-inhib A), although the site is slightly expanded in the K^+^ map, perhaps to accommodate the larger K^+^ ion.
The &beta;6-&beta;7 linker has high quality density in both maps, supporting an almost-identical (C&alpha; RMSD 1.076&nbsp;&Aring; when the whole &alpha; subunit is aligned by the palm domain) conformation, aside from the blurring of &alpha;R333 across multiple rotamers (@fig-sodium-self-inhib B).
Although SSI is sensitive to pH, the buffer conditions for the CKO map should be at least 55% inhibited[@collier2009].
Attempts at detecting other classes of particle in the CKO ENaC (Na^+^) map were not successful, leading us to believe that the particles are homogeneous.
The lack of any significant structural difference in the acidic pocket in the presence of Na^+^ and K^+^ is more compatible with an SSI model in which Na^+^-bound ENaC is "locked" in a closed state, rather than some third inhibited state.

## Glycans make important contacts among subunits and with the membrane

Glycosylation of ENaC at conserved asparagine sites is essential for sodium self inhibition, trypsin activation, and mechanosensitivity[@kashlan2018; @knoepp2020].
The high-quality maps obtained in this study enable modeling of significantly more and longer glycans than in previous work.
I find a total of 19 distinct glycosylation sites, with at most 9 sugars I are able to build into a single glycan.
For the most part, the sugars are identical in all six models and make no protein or glycan interactions.
These sugars likely play a role in other regulatory roles, such as channel trafficking and maturation or interaction with the extracellular matrix[@kashlan2018; @hughey04; @knoepp2020].
However, two classes of glycan are particularly interesting: the micelle-proximal and the intersubunit glycans.

![ENaC's glycosylation sites form important intersubunit interactions. **A**: Cartoon overview of intersubunit glycosylations. Dotted lines represent glycan chains. Hexagons represent sugars, circles represent potentially-interacting residues. Unfiltered maps for the first two sugars are presented to indicate confidence in Asn assignment. **B**: Silver-stained SDS-PAGE gel of protein treated with PNGase F. A shift indicates that the purified protein is glycosylated. **C**: Intersubunit glycan from &beta;N260 to &alpha;D235, K495. **D**: Intersubunit interaction between glycans from &beta;N207 (red) and &gamma;N271 (magenta). In both C and D, the solid surface is a gaussian-blurred version of the sharp human CKO digitonin map (mesh).](figures/intersubunit-glyc/intersub-glyc.png){#fig-intersub-glyc}

The intersubunit glycans stretch from one subunit to another (@fig-intersub-glyc A).
I are able to model long chains for these glycans, with some maps supporting up to nine monosaccharides.
Due to the flexibility of these long chains, the terminal side is quite blurred out in our cryoEM maps, and so I cannot be confident about the exact orientation or identity of the constituent sugars.
Nonetheless, our maps are of sufficient quality to at least suggest sugar-mediated interactions between subunits, and the first sugars are of sufficient resolution to assign the asparagine residues to which they are coupled.
Treatment with PNGase F induces a size shift in our purified protein, confirming the identity of these densities as glycans (@fig-intersub-glyc B)
I built typical high-mannose glycans (Man~5~GlcNAc~2~) into our models as expected for GnTI^-^ and Sf9 cells to assess these interactions at a low resolution[@reeves2002; @shi2007].
One glycan reaches from &beta;N260 to the &alpha; subunit (@fig-intersub-glyc C), potentially hydrogen bonding with &alpha;D235 and &alpha;K459.
These three residues are absolutely conserved across ENaC genes surveyed.
Similarly, glycans attached to conserved sites &beta;N207 and &gamma;N271 connect the &beta; GRIP to the &gamma; palm (@fig-intersub-glyc D).
Knock-out of sugars in similar positions in &alpha;ENaC increase blood pressure in mice[@knoepp2020].
The relatively high quality of these glycans suggest they are quite conformationally stable and may play a role in channel function.

![ENaC's glycans contact the micelle. **A**: An overview of the position of the micelle-proximal glycans and the micelle. The image on the right is a zoomed view of the boxed region on the left **B**:  In-grid rotation distribution for two selected classes of CKO nanodiscs. Note that both classes have even distribution of in-plane angles, indicating that the micelle tilt does not align with some grid force such as fluid flow during blotting. **C**: A cartoon representation of the position of the membrane-proximal glycans relative to the micelle tilt axis. **D and E**: The sharpened map of human CKO digitonin for the first sugar attached to &alpha;N397 and &beta;N364, respectively.](figures/membrane-glyc/membrane-glyc.png){#fig-micelle-glyc}

The micelle-proximal glycans are anchored to h&alpha;N397 and h&beta;N364 and the equivalent mouse residues, m&alpha;N424 and m&beta;N362 (@fig-micelle-glyc A).
The &alpha; site is conserved in mammalian ENaCs, while the &beta; site is conserved in all surveyed organisms except lungfish.
These glycans extend away from the ECD body and turn down toward the micelle or lipids and are clear in all ENaC maps solved to date.
The &gamma; subunit does not have an equivalent asparagine residue, and so does not have a micelle-proximal glycan.
I note that all ENaC maps produced to date also show a consistent tilt of the micelle or nanodisc such that the &gamma; subunit is closer to the surface of the micelle than are the &alpha; or &beta; subunits; this tilt does not appear to be an artifact of in-plane rotation (@fig-micelle-glyc B).
I propose therefore that this tilt may in part be caused by the presence of the micelle-proximal glycans, which are only missing from &gamma;ENaC (@fig-micelle-glyc C).
Their presence in the nanodisc map may indicate that they perform a similar role in a native cell membrane context.


<!-- Move this to the end of content so the margins are right in citations -->
:::{.content-hidden unless-format="pdf"}
\newgeometry{left=1in, right=1in}
:::